\documentclass[12pt]{article}
\usepackage{graphicx}
\usepackage{siunitx}
\usepackage{authblk}
\usepackage{amsmath}
\usepackage{booktabs} % for much better looking tables
\usepackage{array} % for better arrays (eg matrices) in maths
\usepackage{verbatim} % adds environment for commenting out blocks of text & for better verbatim
\usepackage{hyperref}
\usepackage{soul}
\usepackage[table,xcdraw]{xcolor}
\usepackage{url}
\usepackage[parfill]{parskip}
\usepackage[top=1in, bottom=1in, left=0.9in, right=0.9in]{geometry}
\geometry{letterpaper}
\usepackage{mathptmx}
\usepackage{lineno}
\linenumbers
\usepackage{afterpage}
\usepackage[T1]{fontenc}
\usepackage{amsmath}
\numberwithin{equation}{section}
% \usepackage[numbers]{natbib}
% \usepackage{fancyvrb}
%\usepackage{lineno}
\usepackage{cleveref}
\usepackage{xr}
\usepackage{natbib}
\bibliographystyle{abbrvnat}
\setcitestyle{authoryear,open={(},close={)}}
\usepackage{ccaption}% http://ctan.org/pkg/ccaption
\usepackage[labelfont=bf]{caption}

\makeatletter
\newcommand*{\addFileDependency}[1]{% argument=file name and extension
  \typeout{(#1)}
  \@addtofilelist{#1}
  \IfFileExists{#1}{}{\typeout{No file #1.}}
}
\makeatother

\newcommand*{\myexternaldocument}[1]{%
    \externaldocument{#1}%
    \addFileDependency{#1.tex}%
    \addFileDependency{#1.aux}%
}

\myexternaldocument{supplemental-info}

\title{A scalable and automated pipeline for the recovery of eukaryotic MAGs}
\author[1,*]{Harriet Alexander}
\author[2]{Sarah K. Hu}
\author[1,3]{Arianna I. Krinos}
\author[1]{Maria Pachiadaki}
\author[4]{Benjamin J. Tully}
\author[4]{Chris J. Neely}
\author[5]{Taylor Reiter}

\affil[1]{\small{Biology Department, Woods Hole Oceanographic Institution, Woods Hole, MA, USA}}
\affil[2]{Marine Chemistry and Geochemistry, Woods Hole Oceanographic Institution, Woods Hole, MA, USA}
\affil[3]{MIT-WHOI Joint Program in Oceanography, Cambridge and Woods Hole, MA, 02540}
\affil[4]{Department of Biological Sciences, University of Southern California, Los Angeles, CA 90089}
\affil[5]{Population Health and Reproduction, University of California, Davis, Davis, CA, 95616}
\affil[*]{Correspondence; halexander@whoi.edu}

\date{}

\begin{document}

\maketitle

\section*{Abstract}

\section*{Introduction}

Unicellular microbial eukaryotes, or protists, play a critical part in all ecosystems found on the planet. In addition to their vast morphological and taxonomic diversity, protists exhibit a range of functional roles and trophic strategies \citep{Caron2011Marine}. Protists are centrally important to global biogeochemical cycles, mediating the crucial pathways for the synthesis and processing of carbon and nutrients in the environment [citations]. Despite their importance across ecosystems and in the global carbon cycle, research on microbial eukaryotes typically lags behind that of bacteria and archaea \citep{Caron2009Hypotheses, Keeling2017Marine}. Consequently, fundamental questions surrounding microbial eukaryotic ecological function remain unresolved. Novel approaches that enable genome retrieval from environmental -omic data provide a means of bridging that knowledge gap. 

Using associations based on assembled genetic fragments (contigs derived from metagenomic reads), abundances, and tetranucleotide frequency, we can reconstruct likely genomic units (genomes) by sorting contigs into bins \citep{Alneberg2014Binning, Wu2014MaxBin, Kang_2019, Graham2017BinSanity}. These bins are refined through a series of steps to generate metagenome assembled genomes or MAGs  \citep{Parks2017Recovery, Delmont2018Nitrogen-fixing, Tully2018reconstruction, Almeida2019new}. Binning metagenomic data into MAGs has revolutionized how researchers ask questions about microbial communities and has enabled the identification of novel taxa and functional traits \citep{Rinke2019phylogenomic, Tully2019Metabolic}. The recovery of eukaryotic MAGs is not currently commonplace (common practice?) relative to bacteira and arcahea, the reason for this  is arguably twofold: (1) eukaryotic genomic complexity \citep{Zhang2011practical}, complicates both metagenome assembly and MAG retrieval; and (2) there is a bias in currently available metagenomic computational tools towards the study of bacterial and archaeal members of the community. Yet, much can be learned about the diversity and role of eukaryotes in our environment from eukaryotic MAG retrieval \citep{Olm2019Genome-resolved}.

Here we developed and applied EUKHeist, a scalable and reproducible pipeline to facilitate the reconstruction, taxonomic assignment, and annotation of prokaryotic and eukaryotic metagenome assembled genomes (MAGs) from mixed community metagenomes. The EUKHeist pipeline was applied to a metagenomic dataset from the \textit{Tara} Oceans expedition protist-size fractions samples \citep{Carradec2018global}, which encompasses more than 20Tb of raw sequence data. From these large-size fraction metagenomic samples, we recovered over 4,000 prokaryotic MAGs and 900 eukaryotic MAGs. 

\section*{Results and Discussion}

We developed the EUKHeist metagenomic pipeline to automate the recovery and classification of eukaryotic and prokaryotic MAGs from large-scale environmental metagenomic datasets. EUKHeist was applied to the metagenomic data from the large-size fraction metagenomic samples ($0.8\mu m-2000\mu m$)  from \textit{Tara} Oceans \citep{Carradec2018global}, which is dominated by eukaryotic organisms. We generated 94 co-assembled metagenomes based on the ocean region, size fraction, and depth of the samples (\Cref{fig:tara-map}), which totaled 180 Gbp in length (Supplemental Table 1).  From these metagenomes we recovered a set of 988 eukaryotic MAGs and 4,022 prokaryotic MAGs, which we are calling \textit{Tara} Oceans Particle Associated MAGs (TOPAZ). [] The TOPAZ MAGs expand the current repertoire of eukaryotic genomic references for the marine environment and shed light on the biogeographical and functional potential of these eukaryotic-dominated marine communities. 

\subsection*{Eukaryotic genome recovery from metagenomes covers major \\ eukaryotic supergroups}

% \afterpage{\clearpage}
\begin{figure}[h!]    
    \centering
    \includegraphics[width = \columnwidth]{figures/Figure3_EukPhylogeny_30Busco-v2-01.png}
    \caption{[Continued on next page.]}
    \label{fig:fig3-euk}
\end{figure}
\begin{figure}[t]
  \contcaption{\textbf{TOPAZ eukaryotic MAGs span the eukaryotic tree of life.} The maximum likelihood tree was inferred from a concatenated protein alignment of 49 proteins from the eukaryotic BUSCO gene set that were found to be commonly present across at least 75\% of the 485 TOPAZ eukaryotic MAGs that were estimated to be >30\% complete based on BUSCO ortholog presence. Branches (nodes) are colored based on consensus protein annotation estimated by EUKulele and MMSeqs. The Ocean Region (OR), Depth (D), and Size Fraction (SF) of the co-assembly that a MAG was isolated from is color coded as colored bars. The completeness (comp) and contamination (cont) as estimated based on BUSCO presence are depicted as a heatmap. Predicted Heterotrophy Index (H-index), which ranges from phototroph-like (-300) to heterotroph-like (300) is shown as a heatmap. The predicted trophic mode (T-pred) based on the trophy random forest classifier with heterotroph (pink) and phototroph (green), is depicted. The number of proteins predicted with EukMetaSanity are shown as a bar graph along the outermost ring. }% Continued caption
\end{figure}

The EUKHeist classification pipeline identified 988 putative eukaryotic MAGs following the refinement of recovered metagenomic bins based on length ($>2.5$ Mbp) and proportion of base pairs predicted to be eukaryotic in origin by EukRep (\Cref{fig:recovered}). Protein coding regions were predicted with the EukMetaSanity pipeline (CITATION), and the likely taxonomic assignment of these bins was made with MMSeqs  \citep{Steinegger2018} and EUKulele \citep{Krinos2021EUKulele} (Supplemental Dataset 2). Of the 988 eukaryotic MAGs recovered, 713 MAGs were estimated to be more than 10\% complete based on presence of core eukaryotic BUSCO orthologs. For the purposes of our subsequent analyses, we only consider the highly complete eukaryotic TOPAZ MAGs, or those that were greater than 30\% complete based on BUSCO ortholog presence (n=485) (\Cref{fig:fig3-euk}). 

Eukaryotic genomes are known to be both larger have higher proportions of non-coding DNA than bacterial genomes \citep{Zhang2011practical}. 
On average across sequenced eukaryotic genomes, 33.1\% of genomic content codes for genes (2.6\% - 59.8\% for the 1st and 3rd quarterlies) \citep{Hou_2009}, while bacterial genomes have a higher proportion of coding regions (86.9\%; 83.9\% - 89.3\%) \citep{Hou_2009}. The high-completion TOPAZ eukaryotic MAGs have an average of $73.7\% \pm 14.3\%$ gene coding \Cref{fig:hq-prot-bar}. This trend of a higher proportion of coding regions was consistent across eukaryotic groups, where Haptophyta and Ochrophyta TOPAZ MAGs had an average coding region of $80.3 \pm 4.9\%$  and $78.1 \pm 6.3\%$, respectively. Genomes from cultivation efforts from Haptophyta (\textit{Emiliania huxleyi} CCMP1516 with 31 Mb or 21.9\% \citep{Read2013}), and Ochrophyta (\textit{Phaeodactylum tricornutum} with 15.4 Mb or 57.3\% \citep{Bowler_2008}) had significantly lower proportions of their genomes coding for proteins than TOPAZ MAGs. The lowest percentages of gene coding were within Metazoan and Fungal TOPAZ MAGs, with $52.6\pm 9.8\%$ and  $58.8 \pm 6.7\%$, respectively. As a point of comparison, the human genome is estimated to have $\approx 34$ Mb or $\approx 1.2$\% of the  genome coding for proteins \citep{humangenome}. Globally, the higher gene coding percentages for the recovered eukaryotic TOPAZ MAGs likely reflect biases caused by the use of tetranucleotide frequencies in the initial binning \citep{Kang_2019} as well as challenges inherent in the assembly of non-coding and repeat-rich regions of eukaryotic genomes. 

%Euk
% 75%, 59.68586387434556
% 50%, 33.1588132635253
% 25%, 2.617801047120425
% 10%, 1.3961605584642416
% 90%, 81.15183246073298
% bact
% 50, 86.9109947643979
% 25, 83.94415357766142
% 75, 89.35427574171028
% 90, 91.09947643979056
% 10, 79.23211169284467


% tree covers all branches of the euk tol, retrieved from all ocean regions.
Phylogenetic placement of TOPAZ MAGs aligned with estimated taxonomy based on protein-consensus annotation (\Cref{fig:fig3-euk}). The recovered MAGs spanned 8 major eukaryotic supergroups: Archaeplastida (Chlorophyta), Opisthokonta (Metazoa, Choanoflagellata, and Fungi), Amoebozoa, Apusozoa, Haptista (Haptophyta), Cryptista (Cryptophyta), and the SAR supergroup (Stramenopiles, Alveolata, and Rhizaria) \citep{Burki_2020}. Eukaryotic MAGs were retrieved from all ocean regions surveyed, with the largest number of high-completion TOPAZ MAGs recovered from the South Pacific Ocean Region (SPO) (n=143) and the fewest recovered from the Southern Ocean (SO) (n=11) and Red Sea (RS) (n=12) (\Cref{fig:hq-dist}). These regional trends in MAG recovery and taxonomy aligned  with the overall sequencing depth at each of these locations (Supplemental Dataset 1), with fewer and less diverse MAGs recovered from the SO and RS (\Cref{fig:all-dist,fig:hq-dist}).

% Size fraction breakdown, start with smallest, others dom by Metaz - smaller
The largest number of MAGs was recovered from the smallest size fraction ($0.8-5 \mu m$) (n=311) (\Cref{fig:fig3-euk,fig:all-dist}), and yielded the highest taxonomic diversity, including MAGs from all the major supergroups listed above (\Cref{fig:all-dist}). Chlorophyta (n=133), Ochrophyta (n=57), and taxa placed within the SAR group (Stramenopiles, Alveolata, and Rhizaria) (n=56) made up the  the largest proportion of small size fraction MAGs. Chlorophyta MAGs were smaller and had fewer predicted proteins relative to other eukaryotic MAGs, despite demonstrating comparable completeness metrics; the average Chlorophyta MAG size was 13.9 Mbp with 7525 predicted proteins (\Cref{fig:hq-prot-bar}). By contrast, Cryptophyta and Haptophyta had the largest average MAG size with 50.8 Mbp and 44.4 Mbp with an average of 23500 and 24400 predicted proteins, respectively (\Cref{fig:hq-prot-bar}). Fewer eukaryotic MAGs were recovered from the other size fractions $5-20\mu m$ (n=20), $20-180 \mu m$ (n=87), and $180-2000\mu m$ (n=39) (\Cref{fig:all-dist}), and metazoa constituted the most recovered MAGs within these size fractions. Metazoan MAGs had the lowest average completeness ($50 \pm 13\%$) (\Cref{fig:all-prot-bar,fig:hq-prot-bar}); where the average size of recovered metazoan MAGs was 43.2 Mb (6.5-177Mbp), encompassing an average of 14600 proteins (\Cref{fig:hq-prot-bar}). 76 of the 123 metazoan MAGs likely belong to the  Hexanauplia (Copepoda) class. Copepods have been estimated to be up to 2.5 Gb with high variation (10-fold difference) across sequenced members \citep{Jorgensen_2019}. 



% Sample depth breakdown - - discussion of fungi (repeatitive)
MAGs were also retrieved from all discrete sampling depths surface, SRF (n=315), deep chlorophyll max, DCM (n=133), mesopelagic, MES (n=13), as well as samples with no discrete depth, MIX (n=21) and the filtered seawater controls, FSW (n=3). Notably, the FSW included 1 Chlorophyta MAG (TOPAZ\_IOF1\_E003) that was estimated to be 100\% complete with no contamination (Supplemental Dataset 2). Globally, fungal MAGs had the highest overall average completeness ($87 \pm 15\%$) (\Cref{fig:all-prot-bar,fig:hq-prot-bar}).  A total of 16 highly complete fungal MAGs were also recovered, of those, 11 originated from the MES (\Cref{fig:fig3-euk,fig:hq-dist}). Putative fungal TOPAZ MAGs were recovered from the phyla Ascomycota (n=10) and Basidiomycota (n=1) and ranged in size from 12.5-47.8 Mb (\Cref{fig:hq-prot-bar}), which are within range of known average genome sizes for these groups 36.9 and 46.5 Mb, respectively \citep{Mohanta_2015}. 
Relative to other depths, the composition of the TOPAZ MAGS from the basin-scale mesopelagic co-assemblies recovered a higher percentage of fungi. This is similar to other mesopelagic and bathy pealgic molecular surveys where the biomass of fungi outweigh OTHER \citep{Morales_2019,Pernice_2015,Edgcomb_2010}.

% The origin and recovery of fungal TOPAZ MAGs aligned with previous molecular taxonomic estimates \citep{Morales_2019,Pernice_2015,Edgcomb_2010}, where fungi from the deep-sea fungi include functional profiles that are enriched for the metabolism of secondary metabolites, vitamins (e.g. vitamin B5), and amino acids \citep{Morales_2019,DAMARE2006172} 


The metagenome read recruitment to these TOPAZ MAGs paralleled MAG recovery, where metazoan MAGs dominated the larger size fractions ($20-180 \mu m$ and $180-2000 \mu m$) across both the surface and DCM for all stations, and chlorophyta MAGs were dominant across most of the small size fraction stations ($0.8-5 \mu m$) (\Cref{fig:map}). A notable exception are the stations from the Southern Ocean, where Haptophyta and Ochrophyta were most abundant in all size fractions. Compared to the samples from the photic zone, SRF and DCM, the average recruitment of reads from the MES was far lower ($24500 \pm 34450$ average CPM in the MES compared to $131000 \pm 104000 $ and $136000 \pm 85000$ for the SRF and DCM, respectively (\Cref{fig:map}). This suggests that the mesopelagic have high variability across communities \citep{Pernice_2015} and that we did not fully capture the eukaryotic MAGs that adequately describe all surveyed communities. Alternatively, this might suggest that the communities sampled were dominated by prokaryotic community members \citep{Pernice_2014}.  


\subsection*{Trophic mode can be predicted from MAG gene content}

\begin{figure}[h!]    
    \centering
    \includegraphics[width = 0.9\columnwidth]{figures/Figure4_Trophic_Mode_v2-01.png}
    \caption{\textbf{Estimated trophic status of TOPAZ eukaryotic MAGs.} (Top) Trophic status was predicted for each high-completion TOPAZ eukaryotic MAG using a Random Forest model trained on the presence and absence of KEGG orthologs and is shown as a color (green, phototroph, pink, heterotroph). The Heterotrophy Index (H-index) (\cref{eq:hind}) for each MAG is plotted with a box plot showing the range of the H-ind for each higher level group. (Bottom) The relative distribution and abundance of Phototroph (green), non-Metazoan Heterotroph (Pink), and Metazoan Heterotroph (Purple) is depicted across all surface samples. Plots are subdivided by size classes. }
    \label{fig:fig4-trophy}
\end{figure}

Eukaryotic microbes can exhibit a diversity of functional traits and trophic strategies in the marine environment \citep{Worden2015} \citep{Caron2011Marine}; including, phototrophy, heterotrophy, and mixotrophy. Phototrophic protists are responsible for a significant fraction of the organic carbon synthesis via primary production; these phototrophs dominate the microbial biomass and diversity in the sunlit layer of the oceans \citep{Worden2015,de_Vargas_2015}. Phagotrophic protists (heterotrophs), which ingest bacteria, archaea, and smaller eukaryotes, and parasitic protists are known to account for a large percentage of mortality in food webs \citep{Sherr_2002, Caron2011Marine, Worden2015}. Protists are also capable of mixed nutrition (mixotrophy), where a single-cell exhibits a combination of phototrophy and heterotrophy \citep{Stoecker_2017}. Typically, the identification of trophic mode has relied upon direct observations of isolates within an lab setting, with more recent efforts including transcriptional profiling as a means of assessing trophic strategy \citep{Keeling2014, Liu_2016}.  Scaling up these culture-based observations to environmentally-relevant settings \citep{Alexander2015a, hu2018,Gong_2016} has been an important advance in the field for exploring complex communities without cultivation. An outcome from these studies has been the realization that trophic strategies are not governed by single genes \citep{Labarre_2020}, in reality, trophic strategy will be shaped by an organisms' physiological potential and environmental setting. Therefore larger genomic and transcriptomic efforts to predict or characterize presumed trophic strategies among mixed microbial communities will greatly contribute to our understanding of the role that microorganisms play in global biogeochemical cycles. 

Large scale omic approaches, such as the recovered TOPAZ MAGs, because they provide a vast amount of data that can be leveraged alongside presently available reference data to ultimately enable the prediction of complex traits (such as trophic mode) without \textit{a priori} information.  Machine learning (ML) applications can be implemented to access the potential of these large datasets. ML approaches have been recently shown to be capable of accurate functional prediction and cell type annotation using genetic input, in particular for cancer cell prediction \citep{shipp2002diffuse,bashiri2017improving,tabl2019machine}, and functional gene and phenotype prediction in plants \citep{mahood2020machine}. Recently, these approaches have been applied to culture and environmental transcriptomic data to predict trophic mode using currently available trophy annotations \citep{lambert2021dynamic, burns2018gene,Jimenez_2021}. Here, we apply an independent machine learning model to the eukaryotic TOPAZ MAGs to predict their trophic strategies.

Using a reference set built from protistan transcriptomic data, we predicted the trophic mode of the highly complete TOPAZ MAGs using machine learning and direct estimation via presence of important KEGG pathways (\cref{eq:hind}). As the gradient of trophic modes among protists is not strictly categorical, we calculated a Heterotrophy Index (H-index) that places the TOPAZ MAGs on a scale of highly phototrophic (negative values) to highly heterotrophic (positive values) (\Cref{fig:fig3-euk, fig4-trophy}). Thus, for all TOPAZ MAGs we have predicted both a gross trophic category (heterotrophic, mixotrophic, or phototrophic) as well as the quantitative extent of heterotrophy (H-index, \cref{eq:hind}). Broadly, the trophic predictions aligned well with the putative taxonomy of each MAG (\Cref{fig:fig3-euk, fig:fig4-trophy}). For example, TOPAZ MAGs that had taxonomic annotation of well known heterotrophic lineages (Metazoa, Fungi),  were predicted as heterotrophs based on our model.  Further, our data-driven trophic mode predictions correlate well with an independent model designed to identify the presence of photosynthetic machinery and capacity for phagotrophy \citep{burns2018gene} (\Cref{fig:mag-burns,fig:mmetsp-burns}). 



Despite evidence that many lineages recovered include known mixotrophs, no TOPAZ MAGs were identified as mixotrophic using this approach. However, the utility of the H-index enables us to still consider mixotrophic-capable MAGs. We explore the likely reasons for this more deeply in the \Cref{limitations}, but one potential explanation is that MAG recovery targets the genomic content of a eukaryotic lineage; and the evolutionary history of phototrophy and heterotrophy is complicated and varies with respect to species \citep{Flynn_2019}. Therefore, the genetic composition of MAGs may reflect encoded metabolisms that are not necessarily exhibited \texti{in situ}. Another explanation is that mixotrophy is not a singular trait, but rather a spectrum of nutritional abilities that are largely driven by the microorganisms' nutritional needs and surrounding environment. Continued culturing combined with large-scale omic efforts will continue to improve such ML models focused on complex traits and ultimately our ability to predict trophic mode. We suggest that the integration of metagenomic and metatranscriptomic datasets might better reflect the active strategies being used. 


\subsection*{Ecological niches of heterotrophic and phototrophic stramenopiles using metatranscriptomic evidence}

24 high-completion TOPAZ eukaryotic MAGs were subset as a case study to explore the utility of pairing metatranscritpome data with MAG results to characterize the ecological context of less resolved protistan lineages. Selected MAGs included 11 taxonomically-assigned as Dictyochophyceae (silicoflagellates), which were putatively classified as phototrophs with our trophic model (\Cref{fig:fig4-trophy}), and 13 MAGs within a phylogenetically-related stramenopile clade and classified as heterotrophs (\Cref{fig:fig3-euk,fig:fig5-dicty}a). 


\begin{figure}[h!]    
    \centering
    \includegraphics[width = \columnwidth]{figures/Figure5-panel-plot-stramenopile-H.png}
    \caption{ \textbf{Dictyochophyceae and stramenopile MAGs} A subset of 24 highly-complete TOPAZ MAGs taxonomically classified as stramenopiles and Dictyochophyceae. (a) Cluster dendrogram derived from the presence or absence of orthologs grouped MAGs by predicted trophic mode (green indicates phototrophy and pink indicates heterotrophy). Pie charts to the left of each TOPAZ MAG name indicate the relative CPM abundance of each MAG for (left to right pies) ocean region, depth sampled, and size fraction. Bubble plots to the right of each TOPAZ MAG name indicate the total MAG CPM abundance. (b) Summary of shared and unique orthologs based on occurrence in phototrophy- and heterotrophy-predicted TOPAZ MAGs. Venn diagrams indicate category of orthologs shown in each panel, while panels report the KEGG module and prevalence among MAGs in each category (bar plot). (c) Principle component analysis derived from metatranscriptome reads, from the surface and smallest size fraction, mapped to shared orthologs (Shared in all MAGs in (b)) among all 24 MAGs. Symbol size designates Heterotrophy Index, while symbol color denotes predicted trophic mode. 
    }
    \label{fig:fig5-dicty}
\end{figure}
All 24 MAGs had a cosmopolitan distribution primarily originating from surface samples from the smallest size fraction, but some individual MAGs had relatively higher CPM abundances suggesting environmental selection based on oceanographic region (\Cref{fig:fig5-dicty}a, \ref{fig:dictyocho-SAR-map}). The biogeographic patterns of the Dictyochophyceae aligned with existing literature, where Dictyochophyceae are globally distributed and typically found in the euphotic layer of the world ocean (Vaulot et al. 2008; Obiol et al. 2020 - Massana 2011?). Global sampling efforts have also recovered genetic signatures of Dictyochophyceae as a prominent, but not abundant member of the stramenopile group (Obiol et al. 2020, Giner?? tara??) that does not demonstrate remarkable seasonality (Giner et al. 2019). While grazing on bacteria and picocyanobacteria has been observed within mixotrophic Dictyochophyceae, previous work to quantify grazing rates were unsuccessful to due to the low cell abundance (Unrein et al. 2014). Therefore, metagenomic and metatranscriptomic datasets are well suited to illuminate the biogeography and functional potential of the Dicytochophyceae.


% Targeted = looking at things we know
% Unt = taking tha tinformation from the ML and looking at those without apriori knowledge. 

We took both a targeted and untargeted approach to investigate the physiology of these SAR MAGs. First, we assessed ortholog co-occurance across the MAGs to identify genes that are core to all of the SAR MAGs, present wthin only photo (Phototrophy Exclusive), etc. 

Among those genes detected as Phototrophy Exclusive (\Cref{fig:fig5-dicty}b) were transcripts related to chlorophyll biosynthesis (Por; protochlorophyllide reductase), transcripts integral to the pentose phosphate pathways, and acyltransferases and carboxylases associated with de novo fatty acid biosynthesis functions (e.g., ACACA; acetyl-CoA carboxylase biotin, ACSS; acetyl-coA synthetase). Inversely, genes detected as Heterotrophy Exclusive included enzymes that are integral for the breakdown of large sugar molecules such as glycosaminoglycans (e.g., IDUA; L-iduronidase, UDP-glucose:O-linked fucose beta-1,3-glucosyltransferase, NAGLU; alpha N acetylglucosaminidase, IDS; iduronate-2sulfatase, and GALC; galactosylceramidases). Enzymes associated with glycosaminoglycan metabolism may be associated with cell adhesion or the intracellular re-processing of glycosaminoglycan; the latter of which may be a genetic attribute for more heterotrophic lifestyles. 

Ordination results based on transcripts shared across all 24 MAGs (Figure 5c; based on ‘shared in all MAGs’ in Figure 5b) clustered by H-index (symbol size) and TOPAZ MAG identity (Supplementary figure PCoA). While there was some overlap among MAGs predicted to be heterotrophic versus phototrophic, trends dictating the PCA results appeared to be driven by the trophic mode of individual MAGs, rather than region sampled (Figure 5c; SI figure PCA by mag). Further, ordination results resembled previously observed trends from transcript-based efforts to separate phototrophic, mixotrophic, and heterotrophic protistan species from cultivation (Koid et al. Bessier et al.) and the environment( Hu et al.). This demonstrates the value of large untargeted genetic approaches to gain insight into the in situ metabolisms of less explored branches of the eukaryotic tree of life. Dictyochophyceae and heterotrophic stramenopiles have previously been reported in meta’omic analyses of the \textit{Tara} Oceans data (Carradec et al. 2018, Pierella Karlusich et al. 2020; Vorobev et al. 2020, Sieraki et al.). Paired metagenomic and metatranscriptome results, alongside the environmental context provided by a large-scale global sampling effort, and the predicted nutritional strategies we can gain a more comprehensive understanding of protists in our oceans.


\subsection*{Prokaryotic MAGs recovered distinct from previous marine MAG efforts}

\begin{figure}[h!]    
    \centering
    \includegraphics[width = 0.95\columnwidth]{figures/Figure1-TOPAZ-HQ-NR_bac.png}
    \caption{\textbf{Diversity of the high-quality prokaryotic TOPAZ MAGs.} }
    \label{fig:fig1-prokaryotes}
\end{figure}

The vast majority of the retrieved prokaryotic MAGs belonged to Bacteria. The high-quality non-redundant TOPAZ (HQ-NR-TOPAZ) MAGs were comprised of 711 bacterial and 5 archaeal MAGs belonging to 30 different phyla (\Cref{fig:fig1-prokaryotes} Table SX).Fifteen additional phyla were recovered in the medium quality MAGs. Of the 716 HQ-NR-TOPAZ MAGs, 507 were unique based on a 99\% ANI binning from MAGs generated from previous binning efforts from \textit{Tara} Oceans metagenomic data, including \citet{Delmont2018Nitrogen-fixing} (TARA), \citet{Tully2018reconstruction} (TOBG), and \citet{Parks2017Recovery} (UBA) ((\Cref{fig:fig1-prokaryotes}). We also quantified the phylogenetic diversity captured by the TOPAZ MAGs by comparing them to a "neutral" reference set of genomes that approximate the state of marine microbial genomes, dominated by isolate genomes, just prior to incorporation of the \textit{Tara} Oceans derived MAGs. To quantify the novelty of the TOPAZ MAGs, they were phylogenetically assessed relative to a 'neutral' reference set of marine genomes (largely from isolates), which excluded the \textit{Tara} Ocean derived MAGs. Relative to the "neutral" genomic references, the TOPAZ medium quality, non-redundant (MQ-NR) set represented a 42.8\% phylogenetic gain (as measured by additional branch length contributed by a set of data) and 59.9\% phylogenetic diversity (as measured by the total branch length spanned by a set of taxa), as compared to efforts focused solely on the smaller size fractions such as TARA and UBA, which had a smaller degree of gain (31.0\% and 25.8\%, respectively) and diversity (44.4\% and 40.5\%, respectively). When building an inclusive tree that contains the "neutral" reference and all \textit{Tara} Oceans MAGs (TOBG + UBA + TARA + TOPAZ), the TOPAZ MQ-NR MAGs, which are likely particle associated, represented 14.4\% of the phylogenetic gain and 44.7\% phylogenetic diversity suggesting that the TOPAZ MAGs offer the largest increase in phylogenetic novelty when compared to MAGs reconstructed from the metagenomes of the smaller size fractions ($<5.00 \mu m$). This suggests that our current databases are largely skewed towards free-living bacteria, while the complexity of the host- and particle-associated bacterial communities, such as those represented by the TOPAZ MAGs, may be overlooked. 

To confirm the hypothesis that the prokaryotic TOPAZ MAGs included particle-associated members, we examined the genomic features of several selected groups that were well-recovered here and in single-cell amplified genomic datasets (e.g., GORG) \citep{Pachiadaki_2019}.  To avoid potential biases related to completeness and contamination of the genomes, only the HQ-NR MAGs were compared to the GORG SAGs. For the most abundant phyla within these two datasets (Bacteriodota, Cyanobacteria, and Proteobacteria), the average GC\% and estimated genome size of the TOPAZ MAGs were significantly higher than the ones typically reported in free-living marine bacteria, while the TOPAZ MAGs appeared to be enriched in Carbohydrate-Active Enzymes (CAZy) and peptidases (\Cref{fig:fig6-prokfunc}). The HQ-NR TOPAZ MAGs were also found to encode on average more tRNAs per genome (39.5 vs 30). Larger genomes have been considered diagnostic for a copiotrophic lifestyle in bacteria(?) (REF), since the more extended and flexible gene repertoire can facilitate substrate catabolism in organic rich niches such as particles. On the contrary, the streamlined genomes of SAR11 and other groups that have free-living oligotrophic lifestyle require fewer resources to maintain and replicate their genomes and have higher carbon-use efficiency (REF). Similarly, G and C have  have higher energy cost of production and more limited intercellualar availability compared to A and T (REF). The genomic trends observed suggest that TOPAZ populations likely encompass both particle associated and free-living microbes. 

\begin{table}[]
\caption{\textbf{Phylogenetic diversity and gain of various MAGs originating from \textit{Tara} Oceans.} Phylogenetic diversity and gain of prokaryotic MAGs was assessed for this study (TOPAZ), TOBG \citep{Tully2018reconstruction}, UBA \citep{Parks2017Recovery}, and TARA \citep{Delmont2018Nitrogen-fixing} relative to each other as well as a "Neutral" tree comprised of relevant marine bacteria.}
\centering
\begin{tabular}{|
>{\columncolor[HTML]{EFEFEF}}c |c|c|c|c|}
\hline
{\color[HTML]{000000} \textbf{Base tree}}                  & \cellcolor[HTML]{EFEFEF}\textbf{\begin{tabular}[c]{@{}c@{}}MAGs \\ of interest\end{tabular}} & \cellcolor[HTML]{EFEFEF}\textbf{\begin{tabular}[c]{@{}c@{}}No. of \\ MAGs\end{tabular}} & \cellcolor[HTML]{EFEFEF}\textbf{\begin{tabular}[c]{@{}c@{}}Phylogenetic \\ diversity* \end{tabular}} & \cellcolor[HTML]{EFEFEF}\textbf{\begin{tabular}[c]{@{}c@{}}Phylogenetic \\ gain\textdegree \end{tabular}} \\ \hline
{\color[HTML]{000000} \textbf{Neutral}} & TOPAZ (MQ, NR) & 1,571            & 59.9\%  & 42.8\%            \\ \hline
{\color[HTML]{000000} \textbf{Neutral}} & TOPAZ (HQ, NR) & 634              & 41.6\%  & 25.8\%            \\ \hline
{\color[HTML]{000000} \textbf{Neutral}} &   TOBG     & 1,974            & 61.3\%  & 46.7\%            \\ \hline
{\color[HTML]{000000} \textbf{Neutral}} & UBA            & 1,052            & 40.5\%  & 25.8\%            \\ \hline
{\color[HTML]{000000} \textbf{Neutral}} & TARA           & 722              & 44.4\%  & 31.0\%            \\ \hline
{\color[HTML]{000000} \textbf{Neutral}} & TOBG + UBA + TARA               & 3,750            & 66.6\%  & 51.8\%            \\ \hline
{\color[HTML]{000000} \textbf{\begin{tabular}[c]{@{}c@{}}Neutral + \textit{Tara} \\ Oceans MAGs\textsuperscript{HQ}\end{tabular}}}  & TOPAZ (HQ, NR) & 634              & 26.1\%  & 6.2\%             \\ \hline
{\color[HTML]{000000} \textbf{\begin{tabular}[c]{@{}c@{}}Neutral + \textit{Tara} \\ Oceans MAGs\textsuperscript{MQ}\end{tabular}}}  & TOPAZ (MQ, NR) & 1,572            & 44.7\%  & 14.4\%            \\ \hline
{\color[HTML]{000000} \textbf{\begin{tabular}[c]{@{}c@{}}Neutral + \textit{Tara} \\ Oceans MAGs\textsuperscript{MQ}\end{tabular}}} & TOBG           & 1,977            & 48.5\%  & 11.1\%            \\ \hline
{\color[HTML]{000000} \textbf{\begin{tabular}[c]{@{}c@{}}Neutral + \textit{Tara} \\ Oceans MAGs\textsuperscript{MQ}\end{tabular}}}  & UBA            & 1,055            & 23.8\%  & 1.6\%             \\ \hline
{\color[HTML]{000000} \textbf{\begin{tabular}[c]{@{}c@{}}Neutral + \textit{Tara} \\ Oceans MAGs\textsuperscript{MQ}\end{tabular}}} & TARA           & 722              & 28.0\%  & 3.4\%             \\ \hline
\multicolumn{5}{l}{\begin{tabular}[c]{@{}l@{}}\textbf{*} total branch length spanned by a set of taxa\\ \textbf{\textdegree} additional branch length contributed by a set of taxa\\ \textsuperscript{HQ} includes Neutral, TOBG, UBA, TARA, and TOPAZ HQ, NR MAGs\\ \textsuperscript{MQ} includes Neutral, TOBG, UBA, TARA, and TOPAZ MQ, NR MAGs
\end{tabular}}                
\end{tabular}
\label{tab:mags}
\end{table}


\begin{figure}[h!]    
    \centering
    \includegraphics[width = 0.75\columnwidth]{figures/Figure2-TOPAZ_prok_features_vsGORG_mod.png}
    \caption{\textbf{Functional enrichment of TOPAZ prokaryotic MAGs.} }
    \label{fig:fig6-prokfunc}
\end{figure}



\subsection*{Environmental factors structure TOPAZ MAG co-occurrence }
To identify communities of organisms that co-occur across the surface ocean we performed a correlation clustering based on the abundances of the eukaryotic TOPAZ MAGS and the HQ-NR-TOPAZ MAGs(\Cref{fig:fig6-network} a). With this correlation analysis, we employed a modularity optimization algorithm \cite{blondel2008fast} to identify distinct communities of co-occuring organisms. This approach identified seven distinct communities that included 379 of the non-redundant TOPAZ MAGs (\Cref{fig:fig6-network} b). The communities were variably connected to each other with  high connectedness \Cref{eq:connectfinal} among Communities 1,2, and 3, with \hl{[ADD NUMBERS]}.  Community 6 (the smallest of the communities, which consisted of a single Bacillariophyta TOPAZ MAG and five distinct \textit{Synechococcus} TOPAZ MAGs) was highly inter-connected (0.733 connectedness), suggesting that members of this community co-occurred across samples with high fidelity. Moreover, this community was distinct and only shared significant connections to Community 7. The other six communities showed lesser degrees of inter-conceitedness (range: $0.181-0.550$; mean: $0.365\pm0.204$).

The seven communities that we identified based on metagenomic abundance correlations were also found to have significant correlations with environmental factors that define the conditions where these communities were more likely to be abundant. For example, Communities 1 and 3 each tended to occur in warmer waters and to be found in longer-lived eddies (according to the calculation by \cite{d2010fluid} as reported in the \textit{Tara} Oceans metadata \cite{taraoceansconsortium2014ross} as ``residence time'') (\Cref{fig:fig6-network} C; Community 1: $\rho=0.274$, $p=1.74e-8$, Community 3: $\rho=0.216$, $p=7.15e-5$). However, these two communities differed both in their association with nutrients and their taxonomic compositions. Community 1 is dominated by metazoa and bacteria and correlates with lower nutrient conditions (nitrate and nitrite: $\rho=-0.234$, $p=1.21e-10$, phosphate: $\rho=-0.269$, $p=1.76e-14$, silica: $\rho=-0.190$, $p=1.02e-6$). By contrast, Community 3 is comprised of phototrophic chlorophytes and bacteria and is more likely to be found in high-nutrient environments (nitrate and nitrite: $\rho=0.230$, $p=3.04e-10$, phosphate: $\rho=-0.266$, $p=3.82e-14$, silica: $\rho=0.182$, $p=4.39e-6$). We found the highest number of significant correlations with environmental variables were found for the the communities that were most taxonomically compact. Communities 5 and 6, which were comprised by SAR and bacteria, and Ochrophyta and SAR, respectively, were also negatively correlated with nutrient concentrations and positively correlated with temperature (Community 5: nitrate and nitrite: $\rho=-0.140$, $p=3.17e-3$, phosphate: $\rho=-0.147$, $p=1.28e-3$, silica not significant, T$_\textrm{pot}$: $\rho=0.134$, $p=5.71e-3$; Community 6: nitrate and nitrite: $\rho=-0.355$, $p=1.68e-26$, phosphate: $\rho=-0.418$, $p=6.01e-38$, silica: $\rho=-0.187$, $p=1.92e-6$; T$_\textrm{pot}$: $\rho=0.243$, $p=7.83e-12$), suggesting oligotrophic leaning communities. By contrast, Community 2, which was also one of the more diverse communities, with representatives from Metazoa, Fungi, Ochrophyta, Haptophta, SAR, Chlorophyta, Apusozoa, and Amoebozoa and the associated bacteria, was significantly positively correlated  with chlorophyll a ($\rho=0.150$, $p=6.17e-4$). 

%Although we reduced the taxonomic redundancy in these communities based on their ANI, we note that many of the strongest co-occurrences tend to be between organisms of similar taxonomy. In many cases, the most specific taxonomic level that could be resolved was identical for many strongly-associated MAGs, but more precise taxonomy (e.g. family, genus, species) could not be determined. 
The network analysis approach highlights putative associations between particular taxa and lifestyles with environmental parameters, but also illuminates the tendency of particular taxa to co-occur (\Cref{fig:fig6-network}). 

For example, we identified that Communities 2 and 7 were comprised of more cosmopolitan taxa that had fewer of their strong correlations realized among members of their own taxonomic group. These communities similarly had fewer strong correlations with environmental parameters than the remaining communities, in particular with key nutrients like nitrate/nitrate, phosphate, and silica. Establishing environmental correlations further illuminated the strong taxonomic linkages of each of the communities determined computationally in the network analysis. 

In particular, for Community 4, we found significant positive correlations with chlorophyll and net primary productivity (Chla: $\rho=0.401$, $p=2.34e-35$, NPP: $\rho=0.166$, $p=1.28e-3$), while we found negative correlations with ``residence time'' ($\rho=-0.347$, $p=1.98e-14$), a measure of whether the community was likely to be found in older versus newly-formed eddies, and with sea surface temperature ($\rho=-0.263$, $p=3.70e-7$. Hence, this community is likely to be found in cold, productive regions. This community is comprised of Chlorophyta, Cryptophyta, Haptophyta, and Ochrophyta, the major groups containing primarily phototrophic eukaryotic microbes. 18 prokaryotic MAGs were also contained in this community, of which 3 were of order Synechococcales within the Cyanobacteria, and 3 were a Deltaproteobacterium of the new phylum Myxococcota. All told, this guild of MAGs comprises likely photosynthesizers often found in cold, but not necessarily nutrient-rich environments. By contrast, Community 1, comprised primarily of Metazoa, correlates negatively with nutrient availability (nitrate/nitrite and phosphate) and positively with temperature, indicating that this community tends to occur in warm and nutrient-poor waters. The 17 bacteria found in this community include \textit{Vibrio}, order Chitinophagales (2), and Opitutales (2), all of which are known to be either facultative or obligate anaerobes, and may gravitate towards locales where heterotrophic eukaryotes typically dominate. Community 3, which contains only chlorophytes and bacteria, correlates positively with nutrients, chlorophyll, and temperature, and negatively with the depth of the oxygen maximum, indicating that this community is strongly linked to regions where nutrients are upwelled. The bacteria in this community are likely to be of phylum Cyanobacteria or Planctomycetota. 

%As taxonomic annotations become more precise with growing available references, it will be possible to more specifically pinpoint the identity of  generalists and specialists within these communities, and to identify their distributions in the global ocean. 

The network analysis and correlation with environmental parameters that we conducted enables a first look at which specific MAGs tend to co-occur and the environmental context of their co-occurrence.



% As taxonomic annotations become more precise with growing available references, it will be possible to more specifically pinpoint the identity of generalists and specialists within these communities, and to identify their distributions in the global ocean. The network analysis and correlation with environmental parameters that we conducted enables a first look at which specific MAGs tend to co-occur and the environmental context of their co-occurrence.

\begin{figure}[h!]    %Place holder figure for right now. 
    \centering
    \includegraphics[width = \columnwidth]{figures/Figure6_Networks.pdf}
    \caption{ \textbf{Distinct communities recovered from the TOPAZ MAGs.} Network analysis performed on association of eukaryotic MAGs with one another and the bacterial MAGs resulted in the identification of seven communities. These communities showed distinct patterns of taxonomic composition (panel B) and were uniquely correlated to environmental parameters (panel C).}
    \label{fig:fig6-network}
\end{figure}


\section*{Conclusion}





\section*{Materials and Methods}

\subsection*{Data acquisition} The metagenomic and metatranscriptomic data corresponding to the size fractions dominated by eukaryotic organisms ranging from ﻿microbial eukaryotes and zooplankton ($0.8 -  2000 \mu m$) as originally published by \citet{Carradec2018global} were retrieved from ﻿European Molecular Biology Laboratory-European Bioinformatics Institute (EMBL-EBI) under the accession numbers PRJEB4352 (large size fraction metagenomic data) and PRJEB6603 (large size fraction metatranscriptomic data) on November 20, 2018. Only samples with paired end reads (forward and reverse) were used in the subsequent analyses (Supplementary Table ERR numbers and input files). After an initial sample-to-sample comparison with sourmash (SI figure sourmash), it was determined that samples largely clustered by depth and size fraction. Samples were grouped for co-assembly by size fraction ($0.8 - 5 \mu m$, $5-20 \mu m$, $20-180 \mu m$, and $180-2000 \mu m$) as per \citet{Carradec2018global}, depth or sample type (surface (SRF), deep chlorophyll maximum (DCM), mesopelagic (MES), mixed surface sample (MIX), and filtered seawater (FSW)), and geographic location (Supplementary Table X). In cases where a sample did not fall directly within one of the size classes, it was assigned to an existing size class based on the upper um limit of the sample. This grouping resulted in the combination of 824 cleaned, paired FASTQ files samples into 94 distinct co-assembly groups, which were used downstream for co-assembly (assembly stats Table X). 

\subsection*{EUKHeist pipeline for metagenome assembly and binning}The metagenomic analysis, assembly, binning, and all associated quality control steps were carried out with a bioinformatic pipeline, EUKHeist, that enables user-guided analysis of stand-alone metagenomic or paired metagenomic and metatranscriptomic sequence data. EUKHeist is a streamlined and scalable pipeline currently based on the Snakemake workflow engine \citep{Koster2012} that is configured to facilitate deployment on local HPC systems.  Supplemental Figure X outlines the structure and outputs of the existing EUKHeist pipeline. EUKHeist is designed to retrieve and identify both eukaryotic and prokaryotic MAGs from large, metagenomic and metatranscriptomic datasets (Supplementary Figure Pipeline). EUKHeist takes input of sequence meta-data, user-specified assembly pairings (co-assembly groups) (Supplementary Table X), and raw sequence files, and returns MAGs that are characterized as either likely eukaryotic or prokaryotic. 

Here, all raw sequences accessed from the EMBL-EBI were quality assessed with FastQC and MultiQC \citep{Andrews2010FastQC}. Sequences were trimmed using Trimmomatic (v. 0.36; parameters: \texttt{ILLUMINACLIP: 2:30:7, LEADING:2, TRAILING:2, SLIDINGWINDOW:4:2, MINLEN:50}) \citep{Bolger2014Trimmomatic}. Passing mate paired reads were maintained for assembly and downstream analyses. Quality trimmed reads co-assembled assembled based on the assembly groups (Supplementary table X) with MEGAHIT (v1.1.3, parameters: \texttt{k= 29,39,59,79,99,119}) \citep{Li2015MEGAHIT}. Basic statistics were assessed for all assemblies with Quast (v. 5.0.2) \citep{Gurevich_2013} (Supplementary Table Y). Cleaned reads from assembly-group-associated metagenomic and metatranscriptomic samples were mapped back against the assemblies with bwa mem (v.0.7.17) \citep{Li2010Fast}. The bwa-derived abundances were summarized with MetaBat2 (v. 2.12.1) script \texttt{jgi\_summarize\_bam\_cotig\_depths} (with default parameters). The output contig abundance tables were used along with tetranucleotide frequencies to associate contigs into putative genomic bins using MetaBat2 (v. 2.12.1) \citep{Kang_2019}. The Snakemake profile used to conduct this analysis is available at \url{https://www.github.com/alexanderlabwhoi/tara-euk-metag}. A generalized version of the Snakemake pipeline (called EUKHeist) that might be readily applied to other datasets is available at \url{https://www.github.com/alexanderlabwhoi/EUKHeist}. MAGs here are subsequently named and referred to as \textbf{T}ara \textbf{O}ceans \textbf{P}artial \textbf{A}ssociated MAGs (TOPAZ) and are individually named based on their assembly group (SI Table Assembly Group naming)

\subsection*{Identification of putative Eukaryotic MAGs} The binning process described above recovered a total of 16,385 putative bins. These bins were screened to identify high completion eukaryotic and prokaryotic bins. All bins were first screened for length, assuming that eukaryotic bins would likely be greater than 2.5Mbp in size (modeled off of the size of the smallest known eukaryotic genome, $\sim 2.3$Mbp \textit{Microsporidian Encephalitozoon intestinalis} \citep{Corradi2010complete}). Bins larger than 2.5Mbp were screened for relative eukaryotic content using EukRep \citep{West2018Genome-reconstruction}, a k-mer based strategy that estimates the likely domain-origin of metagenomic contigs. EukRep was used to classify the relative proportion of eukaryotic and prokaryotic content in each bin in a contig-by-contig manner. This approach identified 907 candidate eukaryotic bins that were greater than 2.5Mb in length and estimated to have more than 90\% eukaryotic content by length. Protein coding domains were predicted in all 907 putative eukaryotic bins using EukMetaSanity. 

\subsection*{Protein prediction in Eukaryotic MAGs with EukMetaSanity} 
\paragraph{Taxonomy.} The MMseqs2 v12.113e3 \citep{Steinegger2017, Steinegger2018, Mirdita2019} \texttt{taxonomy} module (parameters: \texttt{-s 7 --min-seq-id 0.40 -c 0.3 --cov-mode 0}) was used to provide a first-pass taxonomic assignment of the input MAG for use in a downstream element of EukMetaSanity pipeline that requires an input NCBI taxon id or a taxonomic level (i.e. Order, Family, etc.). We created a custom database comprising both OrthoDB \citep{Kriventseva2018} and MMETSP \citep{Keeling2014} protein databases (OrthoDB-MMETSP) that integrates NCBI taxon ids. MMseqs2 was used to query each MAG against the OrthoDB-MMETSP database to identify a first-pass taxonomic assignment. The lowest common ancestor of top scoring hits was identified to provide taxonomic assignment to each candidate eukaryotic bin. The \texttt{taxonomyreport} module generates a taxon tree that includes the percent of MMseqs mappings that correspond to each taxonomic level. A taxonomic identifier and scientific name are selected to the strain level or when total mapping exceeds 8\%, whichever comes first. The assigned NCBI taxon id is retained for downstream analyses. 

\paragraph{Repeats identification.} RepeatModeler \citep{Flynn-RM, RepeatModeler} was used to provide \textit{ab initio} prediction of transposable elements, including short and long interspersed nuclear repeats, as well as other DNA transposons, small RNA, and satellite repeats. RepeatMasker \citep{RepeatMasker} was then used to hard-mask these identified regions, as well as any Family-level (as identified above) repeats from the DFam 3.2 database \citep{Flynn2020}. RepeatMasker commands \texttt{ProcessRepeats} (parameter: \texttt{-nolow}) and \texttt{rmOutToGff3} (parameter: \texttt{-nolow}) were used to output masked sequences (excluding low-complexity repeat DNA from the mask) as FASTA and gene-finding format (GFF3) files, respectively. 

\paragraph{\textit{Ab initio} prediction.}  GeneMark \citep{Lomsadze2005} was used to generate \textit{ab initio} gene predictions with the repeat-masked eukaryotic candidate bin sequences output from the prior step. The GeneMark subprogram ProtHint attempts to use Order-level proteins from OrthoDB-MMETSP database to generate intron splice-site predictions for \textit{ab initio} modeling using GeneMark EP  \citep{Bruna2020}. If ProtHint fails to generate predictions, then GeneMark will default to ES mode. Due to the fragmented nature of metagenomic assemblies, the prediction parameter stringency was drastically reduced relative to what is recommended for draft genome projects (parameters: \texttt{--min\_contig 500 --min\_contig\_in\_predict 500 --min\_gene\_in\_predict 100}). These parameters can be easily modified within the EukMetaSanity config file. GeneMark outputs predictions of protein coding sequences (CDS) and exon/intron structure as GFF3 files. 


\paragraph{Integrating protein evidence.} MetaEuk \citep{LevyKarin2020} was used to directly map the repeat-masked eukaryotic candidate bins sequences against proteins from the OrthoDB-MMETSP database. MetaEuk \texttt{easy-predict} (parameters: \texttt{--min-length 30 --metaeuk-eval 0.0001 -s 7 --cov-mode 0 -c 0.3 -e 100 --max-overlap 0}) used Order-level proteins to identify putative CDS and exon/intron structure. MetaEuk encodes this output as headers in FASTA sequences that are then parsed into GFF3 files. 

\paragraph{Merging final results.} GFF3 output from the previous \textit{ab initio} and MetaEuk protein evidence steps were input into Gffread \citep{Pertea2020} (parameters: \texttt{-G --merge}) to localize predictions from both lines of evidence into a single GFF3 output file.  Each locus was then merged together using a Python \citep{Python} script and the BioPython API \citep{BioPython} within EukMetaSanity. The set of \textit{ab initio} generated exons in each locus is used as a prediction of the underlying exon/intron structure of the gene locus to which it is assigned. If there are any protein-evidence-generated exons present at the same locus, and if the total numbers of exons predicted by each line of evidence have $\geq 70\%$ agreement, \textit{ab initio} generated exons lacking a corresponding protein-evidence-generated exon are removed (the first and last exon(s) of a locus are not removed). Conversely, any protein-evidence-generated exon present that lacks a corresponding \textit{ab initio} generated exon is added to the predicted exon/intron structure. The final gene structure for each locus is then processed into GFF3 and FASTA format.

\subsection*{Functional and taxonomic annotation of eukaryotic MAGs} 

Predicted proteins from EukMetaSanity were annotated for function against protein families in Pfam with PfamScan \citep{Finn2014Pfam} and KEGG using kofamscan \citep{Kanehisa_2019, Aramaki_2019}. The relative completeness and contamination  of each putative Eukaryotic MAG was assessed based on protein content using BUSCO v 4.0.5 against the eukaryota\_odb10 gene set using default parameters \citep{Simao2015BUSCO} and EukCC v 0.2 using the EukCC database (created 22 October 2019 \citep{Saary2020Estimating}). Annotation and completeness assessment were carried out using a EukHeist-Annotate (\url{https://www.github.com/halexand/Eukheist-annotate}). EukCC \citep{Saary2020Estimating} was also used to calculate MAG completeness and contamination. The average completeness across groups increased in all cases with EukCC except for metazoans, which on average had a lower estimated completeness (\Cref{fig:eukcc}). 

The taxonomic affiliation of the high- and low-completion bins was estimated using MMSeqs taxonomy through EukMetaSanity and EUKulele \citep{Krinos2021EUKulele}, an annotation tool that takes a protein-consensus approach, leveraging a Last Common Ancestor (LCA) estimation of protein taxonomy, as well as MMSeqs2 taxonomy module \citep{Steinegger2017, Steinegger2018, Mirdita2019}. Taxonomic level estimation in EUKulele was assessed based on e-value derived best-hits, where percent id was used as a means of assessing taxonomic level, with the following cutoffs: species, >95\%; genus, 95-80\%; family, 80-65\%; order, 65-50\%; class, 50-30\% modeled off of Carradec et al. (2018). All MAGs were searched against the MarMetZoan combining the MarRef, MMETSP, and metazoan orthoDB databases \citep{Johnson2018Re-assembly, Keeling2014, Kriventseva2018, Klemetsen:2017fg}. This databse is available for download through EUKulele. 

\subsection*{Phylogeny of eukaryotic MAGs} 
%A concatenated gene tree of the high-quality TOPAZ eukaryotic MAGs, the TARA MAGs (Delmont), and relevant select reference genomes and transcriptomes (MMETSP).
A total of 49 BUSCO proteins were found to be present across 80\% or more of the highly complete eukaryotic TOPAZ MAGs and were selected for the construction of the tree. Amino acid sequences from all genomes and transcriptomes of interest were collected and aligned individually using mafft (v7.471) (parameters: \texttt{--thread -8 –auto}) \citep{Katoh2013MAFFT}. Individual protein alignments were trimmed to remove sections of the alignment that were poorly aligned with trimAl (v1.4.rev15) (parameters: \texttt{-automated1}) \citep{Capella-Gutierrez2009trimAl}. Protein sequences were then concatenated and trimmed again with trimAl (parameters: \texttt{-automated1}). A final tree was then constructed using RAxML (v 8.2.12; parameters: \texttt{raxmlHPC-PTHREADS-SSE3 -T 16 -f a -m PROTGAMMAJTT -N 100 -p 42 -x 42}) \citep{Stamatakis2014RAxML}. The amino acid alignment and construction was controlled with a Snakemake workflow: \url{https://github.com/halexand/BUSCO-MAG-Phylogeny/}. Trees were visualized and finalized with iTOL \citep{Letunic2016Interactive}. 

\subsection*{Prokaryotic MAG assessment and analysis} 
The 15,478 bins that were not identified as putative eukaryotic bins based on length and EukRep metrics were screened to identify quality prokaryotic bins. The quality and phylogenetic-association of these bins was assessed with a modified version of MAGpy \citep{Stewart2019MAGpy}, which was altered to include taxonomic annotation with GTDB-TK v.0.3.2 \citep{Chaumeil_2019}. Bins were assessed based on single copy ortholog content with CheckM v \citep{Parks2015CheckM} to identify 2 different bin quality sets: 1) high-quality prokaryotic bins (>90\% completeness, <5\% contamination), and 2) medium-quality prokaryotic bins (90-75\% completeness, <10\% contamination). A total of 4022 prokaryotic MAGs met the above criteria. A final set of 2,407 non-redundant MAGs were identified using dRep v2.6.2 \citep{Olm_2017}, which performs pairwise genome comparisons in two steps. First, a rapid primary algorithm, Mash v1.1.1 \citep{Ondov_2016} is applied. Genomes with Mash values equivalent to 90\% Average Nucleotide Identity (ANI) or higher were then compared with MUMmer v3.23 \citep{Mar_ais_2018}. Genomes with ANI $\geq99\%$ were considered to belong to the same cluster. The best representative MAGs were selected based on the dRep default scoring equation \citep{Olm_2017}. Out of the final set of 2,407 non-redundant MAGS, 716 were high-quality. The same pipeline was used to determine the high-quality non-redundant MAGs reconstructed from the \textit{Tara} Oceans metagenomes in previous studies \citep{Tully2018reconstruction, Parks2017Recovery, Delmont2018Nitrogen-fixing}. 

\subsection*{Phylogeny of bacterial non-redundant high-quality MAGs}
Only 5 out of the 716 high-quality non-redundant MAGs were found to belong to Archaea, thus only bacterial MAGs were used for the construction of the phylogenetic tree with GToTree v.1.4.10 \citep{Lee_2019} and the gene set (HMM file) for Bacteria (74 targets). GToTree pipeline uses Prodigal v2.6.3 \citep{Hyatt_2010} to retrieve the coding sequences in the genomes, and HMMER3 v3.2.1 \citep{Eddy_2011} to identify the target genes based on the provided HMM file. MUSCLE v3.8 \citep{Edgar_2004} was then used for the gene alignments, and Trimal v1.4  \citep{Capella_Gutierrez_2009} for trimming. The concatenated aligned is used for the tree constructions using FastTree v2.1 (CITATION; Price 2010). Three genomes were excluded from the analysis due to having too few of the target genes. The tree was visualized using the Interactive Tree of Life (iToL) \citep{Letunic2016Interactive}.

\subsection*{Prokaryote MAG phylogeny comparison} 
A set of 8,644 microbial genomes were collected from the MarDB database \citep{Klemetsen:2017fg}(accessed 31 May 2018) encompassing the publicly available marine microbial genomes. Genomes were assessed using CheckM 
v1.1.1 \citep{Parks2015CheckM}(parameters: \texttt{lineage\_wf}) and genomes estimated to be <70\% complete or >10\% contamination were discarded. The remaining genomes (n = 5,878) were assessed using CompareM v0.0.23
(parameters: \texttt{aai\_wf}; \url{https://github.com/dparks1134/CompareM}) and near identical genomes were identified using a cutoff of $\geq 95\%$ average amino acid identity (AAI) with $\geq 85\%$ orthologous fraction (determined as one standard deviation from the average orthologous fraction for genomes with $97-100\%$ AAI).
Based on CheckM quality, the genome with the highest completion and/or lowest contamination were retained. From the remaining genomes (n = 3,843), all MAGs derived from the \textit{Tara} Oceans dataset, specifically from Tully et al. \citet{Tully2018reconstruction} and Parks et al. \citet{Parks2017Recovery}, were removed. The remaining genomes (n = 2,275) would be used to form the base of a phylogenetic tree representing the available genome diversity prior to the release of previous \textit{Tara} Oceans related MAG datasets \citet{Tully2018reconstruction, Parks2017Recovery, Delmont2018Nitrogen-fixing}, termed the ``neutral`` component of subsequent phylogenetic trees.

For the comparisons, phylogenetic trees were constructed using GToTree v1.4.7 \citep{Lee_2019} (default parameters; 25 Bacteria\_and\_Archaea markers). Any genome added to a tree that did not meet the default 50\% marker presence requirement was excluded from that tree. Five iterations of phylogenetic trees were constructed using the neutral genomes paired with each \textit{Tara} Oceans MAG dataset, the high-quality TOPAZ prokaryote MAGs, and the medium-quality TOPAZ prokaryote MAGs, individually, and two larger trees were constructed containing all neutral genomes and \textit{Tara} Oceans MAGs, with additions of either high- or medium-quality TOPAZ MAGs. Phylogenetic trees were assessed using genometreetk (parameter: \texttt{pd}; \url{https://github.com/dparks1134/GenomeTreeTk}) to determine the phylogenetic diversity (i.e., the total branch length traversed by a set of leaves) and phylogenetic gain (i.e., the additional branch length added by a set of leaves) \citep{Parks2017Recovery} for each set of MAGs compared against the neutral genomes and for the TOPAZ prokaryote MAGs compared against the neutral genomes and the other \textit{Tara} Oceans MAGs.

\subsection*{MAG abundance profiling} 

Raw reads from all metagenomic and metatranscriptomic samples were mapped against the eukaryotic and prokaryotic TOPAZ MAGs to estimate relative abundances with CoverM (v. 0.5.0; parameters: \texttt{-min-read-percent-identity 0.95 -min-read-aligned-percent 0.75  -min-covered-fraction 0 -contig-end-exclusion 75 -trim-min 0.05 -trim-max 0.95  -proper-pairs-only}; \url{https://github.com/wwood/CoverM}). The total number of reads mapped to each MAG was then used to calculate Reads Per Kilobase Million ($RPKM$), where for some MAG, $i$ :  $RPKM_i = {X_i}/{l_iN}10^9$, with $X =$ total number of reads recruiting to a MAG, $l =$ length of MAG in Kb, $N =$ total number of trimmed reads mapping to a sample in millions. We also calculated counts per million (CPM), a normalization of the RPKM to the sum of all RPKMs in a sample. CPM, a modification of transcripts per million (TPM) was first proposed by \citet{Wagner_2012} as an alternative to RPKM that reduces statistical bias. The metric has since been applied to metagenomics data, sometimes called GPM (genes per million) \citep{Gradoville_2017}. 

\subsection*{Nutritional modelling} 

To predict the trophic mode of the high quality TOPAZ eukaryotic MAGs (n=485), a Random Forest model \citep{Breiman_2001} was constructed and calibrated using the ranger \citep{Wright_2017} and tuneRanger packages in R \citep{tuneRanger}, respectively. The model was trained using KEGG Orthology (KO) annotations \citep{Kanehisa_2019} from a manually-curated reference trophic mode transcriptomic dataset consisting of the MMETSP \citep{Keeling2014} and EukProt \citep{Richter2020EukProt}. 644 of the transcriptomes in this reference dataset came from the MMETSP (Keeling:2014), after 22 transcriptomes were removed due to low coverage of KEGG and Pfam annotations \citep{Finn2014Pfam}. The remaining 266 came from the EukProt database, after 162 were removed due having fewer than 500 present KOs. Nutritional strategy (phototrophy, heterotrophy, or mixotrophy) was assessed for each reference transcriptome individually (citation?), 25\% of the combined reference transcriptomes were excluded from model training as testing data. 

A subset of KEGG Orthologs (KOs) that were predictive for trophic mode classification was determined computationally with the vita variable selection package in R \citep{Janitza_2016}, which was tested and justified by Degenhardt et al. \citet{Degenhardt_2017}. This process was carried out by the algorithm without regard to the predicted function of the KOs, but we found that many of these KOs were implicated in carbohydrate and energy metabolism, with preference for those KOs that differ strongly between heterotrophs and phototrophs (particularly for energy metabolism; Supplementary Figure \ref{fig:ko-trophy}). The model was built using the selected KOs ($n=1787$ of a total 21585 KOs) with the 75\% of the combined database assigned as training data.

Additionally, we developed a secondary metric for assessing the extent of heterotrophy of a transcriptome or MAG. As opposed to the trinary classification scheme of the Random Forest model, this approach quantifies the extent that the MAG aligns with heterotrophic, phototrophic, or mixotrophic references by assigning a composite score. We calculate the likelihood of vita selected KOs used in the Random Forest model above to be present within heterotrophic, phototrophic, or mixotrophic reference transcriptomes. Three scores ($h$, $p$, $m$), one corresponding to each trophic mode, were hence calculated for each vita-selected  KO ($k$) ($n=1787$). In equation \ref{eq:hks}, $K$ is the number of references the KO was present in for each trophic mode category, while $n$ is the total number of references available for each trophic mode category.

\begin{gather}\label{eq:hks}
    h_k = g\left(\frac{K_\textrm{het}}{n_\textrm{het}}\right)\\
    p_k = g\left(\frac{K_\textrm{photo}}{n_\textrm{photo}}\right)\\
    m_k = g\left(\frac{K_\textrm{mixo}}{n_\textrm{mixo}}\right)\\
    g(a) = \begin{cases}
       a & \textrm{if } a > 0.5 \\
       -(0.5-a) & \textrm{ otherwise} \label{eq:connectfinal}
    \end{cases} 
\end{gather}

If  a given KO occurred in fewer than 50\% of the reference transcriptomes for a trophic mode, it was considered not to be characteristic of that trophic mode and as such the score was transformed ($-(0.5 - h),\ \text{if} \ h<0.5$), to reflect the absence. In the test transcriptome dataset, the ratio-transformed scores were negated when a given KO was absent from the transcriptome. Hence, if for instance a KO was found in 10\% of reference transcriptomes assigned to heterotrophy, and absent in the reference transcriptome, it would receive a score of $-1 * (-(1-0.1)) = 0.9$ for that KO.

The scores for all KOs selected by vita were then used to scale the presence/absence patterns observed across transcriptomes and MAGs. Thus, for each transcriptome or MAG a single score was calculated for each trophic mode heterotrophy ($H$), phototrophy ($P$), and mixotrophy ($M$) for all KOs present within the transcriptome or MAG ($K$):
  \begin{equation}\label{eq:sum}
  H = \sum_{k \in K} h_k \\ 
  \end{equation}
  \begin{equation}
    P = \sum_{k \in K} p_k \\
\end{equation}
  \begin{equation}
    M = \sum_{k \in K} m_k
\end{equation}

These calculated values can then be aggregated to a composite heterotrophy score ($H_{ind}$). The score was computed as follows: 
\begin{equation}\label{eq:hind}
  H_{ind}=
    \begin{cases}
      -1^{(H-P)}\sqrt{(H-P)^2}, & \text{if}\ M-max(H,P)<50, \\
      \frac{{}-1^{(H-P)}\sqrt{(H-P)^2}}{M}, & \text{if} \ M-max(H,P) \geq 50
    \end{cases}
  \end{equation}
  
\subsection*{Ecological analysis of SAR and Dictyochophyceae MAGs}
24 highly-complete TOPAZ eukaryotic MAGs were subset to characterize the biogeography and putative trophic modes of \emph{Dictyochophyceae} and closely-related stramenopiles. MAGs included 11 \emph{Dictyochophyceae} MAGs and 13 MAGs belonging to a phylogenetically similar branch (Eukaryotic SAR; derived from the MMSeqs and EUKulele assignment). Metagenomic CPM abundance of MAGs was used to compare the biogeography and distribution of \emph{Dictyochophyceae} and closely-related stramenopile MAGs. MAGs were further classified based on trophic prediction and heterotrophy score. 

To investigate physiological potential, quality trimmed metatranscriptome reads were mapped using Salmon \cite{Patro2017Salmon} to the 24 MAGs. Output information regarding TPM abundances or anything? Comparison of of the putative metabolic capabilities of the predicted phototrophic versus heterotrophic MAGs was conducted in R, where MAGs were clustered using the average distance between MAGs based on the presence and absence of known orthologs (KO annotations \citep{Kanehisa_2019}). Principle coordinate analysis of center log-ratio transformed TPM abundances of mapped reads from the smallest size fraction ($0.8 - 5.00 \mu m$) of all surface samples revealed the degree of overlap between \emph{Dictyochophyceae} and closely-related stramenopile MAGs. Specific genes shared among all MAGs, shared across predicted trophic modes, and those that were used in the prediction of trophic strategy \citep{} were further targeted.

\subsection*{Network Analysis} 

To identify co-occurring MAGs across the stations surveyed by \textit{Tara} Oceans, the CPM abundance of each highly-complete eukaryotic MAG ($>30\%$ BUSCO completeness) and each non-redundant, highly complete bacterial MAG was assessed at each station at all available depths and size fractions as described above. CPM was used because of the power of this metric for comparing samples directly: the sum of all CPM values per sample will be the same, as sequencing depth is accounted for after gene length. This makes it easier to compare the abundances of MAGs originally recovered from different sites \citep{Gradoville_2017}. A Spearman correlation matrix was generated to identify monotonic relationships between MAGs. Correlations were filtered based first on p-value, using the Šidák correction \citep{Sidak_1967}, a slightly less stringent metric than the Bonferroni correction. The Šidák correlation adjusts for multiple comparisons and is given by $p < 1-(1-\alpha)^{1/n}$, where $n$ is the total number of comparisons, and $\alpha$ is the significance value, in this case 0.05. We considered only those correlations within the 90th percentile of CPM correlations, thus correlations with absolute value less than 0.504 were removed from the analysis. Subsequently, we further filtered interactions to those with coefficient of correlation $>0.70$ for the construction of the network diagram. Because it was expected for several of the eukaryotic MAGs to be closely related (based on ANI), the relationships in the network were further filtered to exclude interactions between MAGs of exceedingly high similarity (having both $99\%$ ANI similarity and $>0.70$ coefficient of correlation in the network analysis). 

The MAGs which were highly similar by ANI similarity were coalesced into clusters. If a MAG was found to belong to a cluster of highly ANI-similar MAGs, its cluster identity rather than the MAG name was used for the remainder of the analysis. ANI-based group members tended to have identical taxonomic classifications: only 2 of 94 clusters had different classifications at the order level per EUKulele (Supplementary Figure \ref{fig:network-support}). We generated a network from this reduced set of labeled interactions (cut off at $>0.70$ coefficient of correlation, focusing on interactions between eukaryotes and prokaryotes or eukaryotes and eukaryotes, and using ANI-based clusters instead of MAG names when applicable) using \texttt{igraph} \citep{igraph,teamr}. Communities of highly associated MAGs were identified using a modularity optimization algorithm introduced in \cite{blondel2008fast} and implemented in \texttt{igraph} \citep{igraph}.

We assessed the connectedness within and between communities by calculating a connectedness metric as follows. For the connectedness within a community (one community to itself), we identified the number of ``dense'' connections by counting up the total number of links found between community members, regardless of how many times the particular MAG had been connected to its own community, and divided that number by the total possible ``dense'', meaning the number of connections which would exist if all community members were connected to all other community members. Between different communities, we defined connectedness by qualifying that a ``connection'' is made the first time each MAG from a given community is linked to another community, and calculated this quantity by dividing the number of realized links between community members by the maximum total size of the two involved communities (Figure \ref{fig:fig6-network}b; Equation \ref{eq:connect}-\ref{eq:connectfinal}). 

\begin{gather}\label{eq:connect}
    C_{x,x} = \frac{\Sigma_{x=1}^{n_x} \Sigma_{y=1}^{n_y} f(x,x)}{\frac{n_x (n_{y-1})}{2}} \\
    C_{x,y} = \frac{\Sigma_{x=1}^{n_x} \Sigma_{y=x+1}^{n_y} f(x,y)}{\textrm{max}(n_x,n_y)} \\
    f(a,b) = \begin{cases}
       1 & \textrm{if } a \textrm{ and } b \textrm{ are connected} \\
       0 & \textrm{ otherwise} \label{eq:connectfinal}
    \end{cases} 
\end{gather}

We calculated Spearman correlation coefficients for the relationship between the abundance of communities between stations and several environmental parameters of interest from the \textit{Tara} Oceans metadata \citep{taraoceansconsortium2014ross} (Figure \ref{fig:fig6-network}). We adjusted the p-value of these comparisons using a Bonferroni adjustment within the statistics package in R \citep{teamr}. 


\bibliographystyle{abbrvnat}

\bibliography{main}


\end{document}



