\documentclass[12pt]{article}
\usepackage{graphicx}
\usepackage{siunitx}
\usepackage{authblk}
\usepackage{amsmath}
\usepackage{booktabs} % for much better looking tables
\usepackage{array} % for better arrays (eg matrices) in maths
\usepackage{verbatim} % adds environment for commenting out blocks of text & for better verbatim
\usepackage{subfig} % make it possible to include more than one captioned figure/table in a single float
\usepackage{url}
\usepackage[parfill]{parskip}
\usepackage[top=1in, bottom=1in, left=0.9in, right=0.9in]{geometry}
\geometry{letterpaper}
\usepackage{mathptmx}
\usepackage{lineno}
\linenumbers
\usepackage[T1]{fontenc}
\usepackage{amsmath}
\numberwithin{equation}{section}
% \usepackage[numbers]{natbib}
% \usepackage{fancyvrb}
%\usepackage{lineno}
\usepackage{cleveref}
\usepackage{natbib}
\bibliographystyle{abbrvnat}
\setcitestyle{authoryear,open={(},close={)}}

\usepackage{pdflscape}
% \newcommand{\beginsupplement}{%
%         \setcounter{table}{0}
%         \renewcommand{\thetable}{S\arabic{table}}%
%         \setcounter{figure}{0}
%         \renewcommand{\thefigure}{S\arabic{figure}}%
%      }
    
\renewcommand{\thetable}{S\arabic{table}}%
\renewcommand{\thefigure}{S\arabic{figure}}%

\title{Supplementary Information for "Eukaryotic genomes from a global metagenomic dataset illuminate trophic modes and biogeography of ocean plankton"}
\author[1,*]{Harriet Alexander}
\author[2]{Sarah K. Hu}
\author[1,3]{Arianna I. Krinos}
\author[1]{Maria Pachiadaki}
\author[4]{Benjamin J. Tully}
\author[4]{Christopher J. Neely}
\author[5]{Taylor Reiter}

\affil[1]{Biology Department, Woods Hole Oceanographic Institution, Woods Hole, MA, USA, 02543}
\affil[2]{Marine Chemistry and Geochemistry, Woods Hole Oceanographic Institution, Woods Hole, MA, USA, 02543}
\affil[3]{MIT-WHOI Joint Program in Oceanography, Cambridge and Woods Hole, MA, 02540}
\affil[4]{Department of Biological Sciences, University of Southern California, Los Angeles, CA 90089}
\affil[5]{Population Health and Reproduction, University of California, Davis, Davis, CA, 95616}
\affil[*]{Correspondence; halexander@whoi.edu}
\begin{document}
\maketitle



\section{Taxonomic group correlations with environmental parameters}
The relative abundance of eukaryotic MAGs across samples was correlated with environmental parameters (\Cref{fig:ind-corr}) and were found to broadly cluster by course taxonomic grouping, as such abundances were summed by broad taxonomic group to assess (\Cref{fig:env-corr}) general trends and correlations in patterns of abundance. Globally, we note that metazoan MAG abundance significantly (Bonferroni corrected $p<0.001$) positively correlated with temperature, salinity, and retention time (defined as the average length of time that a particle has been trapped in an eddy), and significantly negatively correlated with a variety of nutrients (phosphate, nitrate+nitrite, silica), CDOM, and depth (meaning they are more abundant in surface samples). This suggests that the metazoan MAGs we are recovering are likely abundant in the subtropical and tropical waters of the large ocean gyres that are oligotrophic. Notably, Choanozoa and Amoebozoa clustered with Metazoans, forming a distinct cluster from other more typically photosynthetic and planktonic groups. Chlorophyta, Apusozoa, Cryptophyta, Dinoflagellata, Ochrophyta, SAR, and Haptophyta clustered separately from Metazoa and were typified by significant positive correlation with oxygen, \textit{a}, and nitrite (\Cref{fig:env-corr}). Additionally, MAGs recovered that belonged to Ochrophyta were found to be significantly negatively correlated with temperature. This suggests that these MAGs were likely associated with more polar or subpolar regions.

\section{Development and performance of the trophic mode model}

\subsection{Description of the model and Heterotrophy Index}
We used a variable selection algorithm and Random Forest machine learning model framework in order to predict the likely trophic mode of the eukaryotic TOPAZ MAGs described in this study. Transcriptomes from the MMETSP and EukProt were manually-annotated as phototroph, mixotroph, or heterotroph based on the literature (Supplementary Table 5). We tested our model with a randomly subset test set comprised of the 25\% of MMETSP and EukProt transcriptomes \citep{Keeling2014,Richter2020EukProt} that were excluded from the model building procedure. With this test subset we obtained an accuracy of 94.6\% (\Cref{fig:confusionmatrix}), meaning that nearly 95\% of taxonomic annotations derived from the machine learning model aligned with their manually-assigned trophic mode annotation (\Cref{fig:confusionmatrix}).  When applied to the TOPAZ MAGs, all MAGs were either classified as phototrophs or heterotrophs, with none classified as  mixotrophs. This likely reflects that the model was generally conservative when it came to assigning genomes or transcriptomes as mixotrophs (\Cref{fig:mag-burns}). As a consequence, we developed a secondary metric for assessing the extent of heterotrophy in the test genomes and transcriptomes using the KOs selected by the vita selection process ($n=1787$), but instead of using the presence or absence of these KOs as a binary indicator to inform the classification of the MAGs, we used the presence or absence as part of an equation to more sensitively assess the number of KOs present that tend to be indicative or either heterotrophy or phototrophy. The result was the ``Heterotrophy Index'' (H-index), a metric for assessing trophy based on KEGG pathway presence or absence. 

The H-index is a sliding scale that weights the presence of heterotrophy, phototrophy- and mixotrophy-indicative KOs to assess the overall likely trophic state of an organism, and will consequently better show when an organism is more likely mixotrophic or possessing traits from both heterotrophy and phototrophy. Because mixotrophs were less common in our test dataset, there are natural concerns about the skill of the model when it comes to identifying them \citep{vabalas2019machine}.We did not attempt to address the imbalance of the categorical training data in the model as other papers have recently explored \citep{utkin2020imprecise,collins2020training}, hence our model retains the bias of reduced sensitivity when the distribution of the training data categories cannot be compared directly to the ``true'' incidence of mixotrophy among eukaryotic organisms \citep{khalilia2011predicting}. Because the majority of test and training transcriptomes were phototrophs or heterotrophs, it is more conservative for a Random Forest model to assign these modes more frequently. As the TOPAZ MAGs covered lineages with known mixotrophic members \citep{jones2000mixotrophy}, and with comparison and feedback from an alternative trophic model as described in \Cref{burnssec}, we applied the H-index to provide more sensitivity in the identification of likely mixotrophs. In particular, with the Random Forest design, if a MAG has characteristics of both heterotrophs and phototrophs that are present in the training data, but does not align with the limited sample of mixotroph transcriptomes (which is also problematic due to the opportunistic nature of the sampling, see  \Cref{limitations}), these MAGs would be assigned the best guess between phototrophy and heterotrophy, when in reality this combination of traits may indicate some form of mixotrophy. The H-index also serves as a confidence metric for the trophy estimate. For example, a MAG with a large positive  H-index would more confidently be called a heterotroph, as this would indicate a strong frequency and degree of alignment with heterotroph references (and, specifically, alignment with those heterotrophic references for KOs identified by the vita algorithm as important for distinguishing trophic mode within the training set). By contrast, a small (or near zero) positive H-index may be mixotrophic or represent a less complete MAG. 

\subsection{Comparison to Burns and Lambert models}\label{burnssec}
In order to assess the performance of our model, which relies solely on assessment of KOs \citep{Kanehisa_2019} that were determined computationally to be important and assessed for function after the fact (\Cref{fig:ko-trophy}), we applied the model from \cite{burns2018gene} (heretofore referred to as the Burns model) to the same highly complete eukaryotic MAGs, as well as to the MMETSP transcriptomes. This model assigns a score from zero to one for individual characteristics related to trophy, including photosynthetic ability, phagocytosis, and prototrophy \citep{burns2018gene}. Using Hidden Markov Models for selected genes which have known association with the aforementioned trophic strategies, the Burns model instead looks for a set of genes consistent with each trait, to assess the ``completeness'' of the genome or transcriptome with respect to the machinery known to be involved with each function. We found the Burns model results to be consistent with our H-index procedure in the following ways. Among the MMETSP transcriptomes, 93.1\% (n=81) of the transcriptomes with a positive H-index (indicative of net heterotrophy) also had a photosynthesis score of less than 0.5 as assigned by the Burns model, and 90.8\% (n=79) had a photosynthesis score of less than 0.05 via the Burns model (Figure \ref{fig:mmetsp-burns}). Similarly, 87.3\% (n=226)  of MMETSP transcriptomes with a negative H-index (indicative of net phototrophy) had a photosynthesis score of greater than 0.5 as assigned by the Burns model (52.9\% (n=137) had negative H-index and Burns photosynthesis score greater than 0.95). MMETSP transcriptomes with a zero or near-zero H-index, which corresponds to putative mixotrophy, had varying photosynthesis scores according to the Burns model, but were more likely to have high (0.6-1) photosynthesis scores, which is consistent with mixotrophy (Figure \ref{fig:mmetsp-burns}). However, several of the MAGs which were predicted to be mixotrophs by the Random Forest model and were annotated manually as mixotrophs from the available metadata had mid- to high- photosynthesis scores in the Burns model, yet negative-leaning H-index scores (Supplementary Table 10). These transcriptomes also tended to have high (>0.7) phagocytosis predictions per the Burns model, consistent with the presence of genetic resources for both heterotrophic and phototrophic strategies. Broadly, we found that, similarly to our H-index and Random Forest model annotations, the Burns photosynthesis prediction results tended to align with the expected lifestyle of each MAG based on EUKulele \citep{Krinos2021EUKulele} taxonomic annotations, with expected heterotrophs like Amoebozoa, Fungi, and Opisthokonta scoring low on photosynthetic ability, while expected phototrophs like Ochrophyta and Chlorophyta tended to score highly for photosynthetic ability (Figure \ref{fig:mmetsp-burns}). Most disagreement was found within the SAR clade and Crypophyta, wherein a range of photosynthesis scores were found by the Burns model, and sometimes these scores contradicted the annotation found by the H-index and Random Forest model (\cref{fig:mmetsp-burns} and Supplementary Tables 9 and 10). This would indicate potentially cryptic and variable trophic strategies and lifestyles within these annotated groups. 

When split into classes of photosynthetic ability based on Burns model scores (to isolate ``highly'' or ``not at all'' photosynthetic: 0-0.1, 0.1-0.45, 0.45-0.55, 0.55-0.9, 0.9-1), MMETSP transcriptomes with ``no'' photosynthesis according to the Burns model (photosynthesis prediction $<0.1$) had an average H-index of $38.85\pm68.19$, while transcriptomes with ``high'' photosynthesis (photosynthesis prediction $>0.9$) had an average H-index of $-272.34\pm119.94$ (\Cref{fig:burns-cats}). As far as the TOPAZ MAGs, we similarly found that the MAGs predicted to be heterotrophic had low variance in photosynthesis prediction as reported by the Burns model, yet the variability in the photosynthetic prediction was high, in particular among those MAGs of higher (less negative, hence closer to ``mixotrophic'') H-index (\Cref{fig:mag-burns}). 

\subsection{The future of trophic mode models}\label{limitations}
The model we developed relies solely upon references that were derived from expression-level data (transcriptomics). Additionally, we used the entire MMETSP \citep{Keeling2014} and EukProt \citep{Richter2020EukProt} databases with manually assigned trophic strategies based on the literature (Supplementary Table 5). Both of these choices carry issues that may be responsible for our under prediction of mixotrophy. First, as they were transcriptomic datasets, the experimental conditions that were used to generate the reference transcriptome are important, in that if a culture was maintained in phototrophy-favorable conditions (e.g. high light, sufficient inorganic nutrients) as opposed to heterotrophy-favorable conditions (e.g. low light, external carbon source), the transcripts reconstructed from these experiments  may result in a reference that is skewed towards phototrophy or heterotrophy, respectively. Regardless of the conditions in which the organism was grown, it is likely that genes present in the genome of the organism were not recovered by transcriptomics. While this means that important genes related to trophic strategy may be missed from the prediction workflow, this is also exciting, as there is much room for growth for these already accurate and skillful models. As genomic references become available and the number of transcriptomic experiments continue to grow, we can expect to further constrain the classes of genes that decide trophic mode. As databases grow, they may be pruned to include only experiments in which the trophy of the organism was known exactly, which would enable the patterns of expression which decide trophic mode to be  pinpointed more precisely. Ideally, a complete core set of protein families necessary for heterotrophy and for phototrophs could be identified. A fundamental question that remains, however, is if there are any genes or protein families that are characteristic of mixotrophic organisms \textit{only} or if these organisms are better characterized based on the co-occurrence of genes indicative of phototrophy and heterotrophy. If the latter,  mixotrophy would be best characterized by the proportion of these sets which overlap in the genome or the expression profile of the candidate organism. While this is in principle the aim of the Burns model (\cite{burns2018gene}, Section \ref{burnssec}), this approach might still be augmented by machine learning-based techniques that have the capacity to identify important genes that may not have yet been annotated. 

\section*{Supplementary Tables}

All Supplementary tables are available through the open science framework at \url{https://osf.io/twz2f/}. 

\begin{itemize}
    \item \textbf{Supplementary Table 1.} Assembly group description, sample inclusion, and basic assembly statistics. 
    \item \textbf{Supplementary Table 2.} TOPAZ Eukaryotic MAG taxonomy, genomic characteristics (e.g., total length, GC content, N50, number of predicted proteins,), and estimated completeness and contamination. 
    \item \textbf{Supplementary Table 3.} TOPAZ Prokaryotic MAG taxonomy as estimated by GTDB, dataset indication (non-redundant representatives (NR), total size, and estimated completeness and contamination. 
    \item \textbf{Supplementary Table 4.} TOPAZ Prokaryotic MAG summary of recovered MAGs across phyla. Total counts are shown for all MAGs (All), the non-redundant subset of MAGs (AllNR), and the high-quality, non-redundant MAGs (HQNR). 
    \item \textbf{Supplementary Table 5.} Reference transcriptomes used in the construction and testing of the trophic model. The manually annotated trophic status and details, relevant reference literature, and taxonomic information are noted. Additionally, a presence/absence matrix is provided for all eukaryotic KEGG ids considered here. 
    \item \textbf{Supplementary Table 6.} Vita selected KOs and their associated heterotrophy, phototrophy, and mixotrophy ratios as described in equations 1-4. 
    \item \textbf{Supplementary Table 7.}  KEGG presence and absence for the TOPAZ eukaryotic MAGs, as was used in the trophic model. 
    \item \textbf{Supplementary Table 8.} Pfam presence and absence across the TOPAZ eukaryotic MAGs. 
    \item \textbf{Supplementary Table 9.} Eukaryotic TOPAZ MAG predicted trophic status and heterotrophy index (H-index). 
    \item \textbf{Supplementary Table 10.} Burns model \citep{burns2018gene} predicted prototrophy, photosynthetic ability, and phagocytosis for the TOPAZ eukaryotic MAGs. 
    \item \textbf{Supplementary Table 11.} Network analysis community composition. 
    \item \textbf{Supplementary Table 12.} Eukaryotic cluster groups derived from average nucleotide identity clustering of eukaryotic TOPAZ MAGs with the Delmont eukaryotic MAGs \citep{Delmont_2020} based on an ANI cutoff of 99\%. 
    \item \textbf{Supplementary Table 13.} Environmental correlations with network derived communities. 
\end{itemize}

\section*{Supplementary Figures}

\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/Tara_stationMap-01.png}
    \caption{Sample map and distribution of sequence data sets across regions. Stations from \textit{Tara} Oceans were categorized into general regions which are highlighted by color. The depth (surface (SRF), deep chlorophyll max (DCM), mixed surface sample (MIX), mesopelagic (MES), and filtered seawater (FSW)) and size fraction are shown as bar plots for each ocean region. }
    \label{fig:tara-map}
\end{figure}
\end{landscape}

\begin{figure}
    \centering
    \includegraphics[width=0.7\columnwidth]{si-figures/flow-chart-01.png}
    \caption{Flow chart of the EukHeist pipeline.}
    \label{fig:flowchart}
\end{figure}
\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/modified-sourmash-region-size_depth-01.png}
    \caption{Sourmash comparison of all metagenomic samples from \textit{Tara} Oceans large size fraction dataset. A minhash comparison was calculated using sourmash (k=31, scale=10,000) of the 824 metagenomic samples corresponding to the large size fraction metagenomic data from \textit{Tara} Oceans (PRJEB4352) \citep{Titus_Brown_2016}. The relative sequence content similarity is shown as Jaccard similarity. Hierarchical clustering of samples based on sequence content is shown and sample identity (sample depth, size fraction, and ocean region) is highlighted by colored blocks.}
    \label{fig:sourmash}
\end{figure}


\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/MAGS_recovered.png}
    \caption{TOPAZ Recovered Eukaryotic MAGs. Course level taxonomic categorization of recovered eukaryotic TOPAZ MAGs (n=988). For each taxonomic group, the total number of MAGs is depicted. MAGs within a taxonomic group that were highly complete ($>30\%$ BUSCO completeness) are shaded and the percentage of highly complete MAGs for each taxonomic group is reported. }
    \label{fig:recovered}
\end{figure}

\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/ALL_MAG_distributions.png}
    \caption{TOPAZ Eukaryotic MAG as recovered by assembly group. The taxonomic breakdown of eukaryotic MAGs recovered within each general type of assembly group (based on Ocean Region, Depth, and Size Fraction) for all eukaryotic MAGs recovered in this study (n=988). Taxonomy is shown both as a total number recovered (top) and as a proportion of MAGs recovered for a given category (bottom). }
    \label{fig:all-dist}
\end{figure}
\end{landscape}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/ALL_MAGS_Num-Prot-Leng.png}
    \caption{The number of predicted proteins as a function of total MAG length.The number of predicted proteins for each eukaryotic TOPAZ mag (n=988) is plotted against the total MAG length (Mb). Each MAG is colored by its taxonomic group and the size of the circle is scaled by the estimated BUSCO completeness.}
    \label{fig:all-prot-len}
\end{figure}

\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/ALL_MAG_protein_bar_plots.png}
    \caption{Genomic traits of recovered eukaryotic completeness, protein predictions, and annotation of all eukaryotic TOPAZ MAGs (n=988). The number of predicted proteins, proportion coding sequences, proportion repeat content, BUSCO completeness, percent annotation with Pfam and KEGG ontology are shown as box and whisker plots for the major higher-level groups that we define for this paper.}
    \label{fig:all-prot-bar}
\end{figure}
\end{landscape}

\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/HQ_MAG_distributions.png}
    \caption{TOPAZ Highly Complete Eukaryotic MAGs as recovered by assembly group. The taxonomic breakdown of eukaryotic MAGs recovered within each general type of assembly group (based on Ocean Region, Depth, and Size Fraction) for highly complete eukaryotic MAGs ($>30\%$ BUSCO completeness)recovered in this study (n=485)). Taxonomy is shown both as a total number recovered (top) and as a proportion of MAGs recovered for a given category (bottom). }
    \label{fig:hq-dist}
\end{figure}
\end{landscape}


\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/HQ_MAG_protein_bar_plots.png}
    \caption{Genomic traits of recovered eukaryotic completeness, protein predictions, and annotation of the highly complete eukaryotic TOPAZ MAGs (n=485). The number of predicted proteins, proportion coding sequences, proportion repeat content, BUSCO completeness, percent annotation with Pfam and KEGG ontology are shown as box and whisker plots for the major higher-level groups that we define for this paper.}
    \label{fig:hq-prot-bar}
\end{figure}
\end{landscape}


\begin{figure}
    \centering
    \includegraphics[width=0.6\columnwidth]{si-figures/HQ_BUSCO-EukCC-comp.png}
    \caption{A comparison of two metrics of eukaryotic completeness. BUSCO completeness as defined based on the presence and absence of genes within the eukaryota\_odb10 dataset was compared against the estimated EukCC completeness based on estimated lineages of given MAGs. Generally, it was observed that EukCC performed particularly well for certain groups (e.g. chlorophytes and fungi) and less well for others (e.g. metazoa).}
    \label{fig:eukcc}
\end{figure}


\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/Distribution-Map-Taxonomy-01.png}
    \caption{The distribution of the major lineages of eukaryotic TOPAZ MAGs recovered across the \textit{Tara} Oceans metagenomic datasets. The counts per million (CPM) is depicted for all stations depicted by depth and size fraction.  }
    \label{fig:map}
\end{figure}
\end{landscape}

\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/modified-individual-mag-group-correlation-01.png}
    \caption{A Spearman correlation between the metagenomic abundance of each of the 485 high-completion eukayrotic TOPAZ MAGs and environmental parameters from the sampling \citep{taraoceansconsortium2016ecoa}, modeled mesoscale physical features based on \citet{d_Ovidio_2010} (indicated with *) , and averaged remote sensing products (indicated with **).}
    \label{fig:ind-corr}
\end{figure}


\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/mag-group-correlation-01.png}
    \caption{ A Spearman correlation between the summed metagenomic abundance of each taxonomic group and environmental parameters from the sampling \citep{taraoceansconsortium2016ecoa}, modeled mesoscale physical features based on \citet{d_Ovidio_2010} (indicated with *) , and averaged remote sensing products (indicated with **). Significant Spearman correlations, those with a Bonferroni adjusted $p<0.01$, are indicated with a dot on the heatmap.}
    \label{fig:env-corr}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/si-combined-phylogeny-delmont-01.png}
    \caption{A concatenated protein phylogeny containing TOPAZ and Delmont \citep{Delmont_2020} eukaryotic MAGs that were estimated to be greater than 30\% complete as well as reference genomes from cultured isolates. The maximum likelihood tree was inferred from a concatenated protein alignment of 49 proteins from the eukaryotic BUSCO gene set that were found to be commonly present across at least 75\% of the 485 TOPAZ eukaryotic MAGs that were estimated to be >30\% complete based on BUSCO ortholog presence (the same proteins that were used in Figure 1). Branches (nodes) corresponding to TOPAZ MAGs are colored based on consensus protein annotation estimated by EUKulele and MMSeqs. }
        \label{fig:delmontphylo}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/Alexander_Delmont_clustermap-01.png}
    \caption{Average nucleotide identity (ANI) between the High Completion TOPAZ and Delmont MAGs. The ANI as estimated by FastANI between all eukaryotic TOPAZ MAGs and Delmont Eukaryotic MAGs is depicted. The taxonomic affiliation of the TOPAZ MAGs is depicted by color along the x-axis. A clustering was done with a cutoff of 99\% ANI generating 83 unique clusters of MAGs, of which 46 contained a representative from both TOPAZ and Delmont (Supplementary Table 12). }
        \label{fig:delmontfani}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/SI-SAR-Dictyocho-MAG-KEGG-intersection.png}
    \caption{Intersection of shared or unique KEGG IDs across 24 SAR and \textit{Dictyochophyceae} TOPAZ MAGs. Subset of MAGs (named on the left) indicated by dot grid (top; ggupset) and the total number of shared KEGG IDs (bottom) as barplots. The largest 30 sets of shared or unique KEGG ID intersections is shown. Combination of MAG intersections show by colors denote the KEGG module, where NA (white) indicates that the KEGG ID did not belong to a KEGG module. }
        \label{fig:dictyocho-SAR-intersect}
\end{figure}


\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/SI-SAR-Dictyocho-MAG-MAG-relabun-bysample.png}
    \caption{Relative counts per million (CPM) of 25 selected SAR and \textit{Dictyochophyceae} TOPAZ MAGs (x-axis) by sample (y-axis and color bar). CPM of MAGs were summed by sample type, size fraction, depth, and ocean region, and Z-score transformed (heatmap signature) for visualization with pheatmap (R; v1.0.12). MAGs and samples are clustered based on average distance.
    }
    \label{fig:dictyocho-SAR-heatmap}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/SI-stramenopile-MAG-map.png}
    \caption{Metagenomic abundance plots for the 24 SAR and  \textit{Dictyochophyceae} TOPAZ MAGs considered. The relative abundance in CPM is shown as a bubble size. Color indicates the depth of the sample. }
    \label{fig:dictyocho-SAR-map}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/SI-SAR-Dictyocho-PCA-byMAG.png}
    \caption{Principle component analysis derived from metatranscriptome reads, from the surface and smallest size fraction, mapped to shared orthologs (Shared in all MAGs in (Figure 5 b)) among selected 24 TOPAZ MAGs. Symbol size designates Heterotrophy Index, while symbol color denotes each TOPAZ MAG.}
    \label{fig:dictyocho-SAR-PCA-mag}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/SI-horizontal-collapsed-clades-05062021.png}
    \caption{Concatenated BUSCO phylogeny of the 24 selected TOPAZ MAGs with a selection of other stramenopile MAGs and SMAGs from \citet{Delmont_2020}. The phylogeny is pictured with iTOL. Collapsed clades include a mixture of TOPAZ and Delmont-generated MAGs that are listed.}
    \label{fig:dictyocho-SAR-tree-itol}
\end{figure}

\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/SI-phototrophs-lolli-05062021.png}
    \caption{Lollipop plot showing the relative CPM of mapped transcripts associated with more phototrophic (left) or heterotrophic (right) traits for 4 \textit{Dictyochophyceae} MAGs (left to right). Relative abundance is shown by MAG for each sample (y-axis and color bar). Shown TOPAZ MAGs were classified as primarily phototrophic (low H-index, reported next to each TOPAZ MAG name), but relative abundance of mapped transcripts varies by sample type.}
    \label{fig:dictyocho-SAR-pho-lolli}
\end{figure}
\end{landscape}

\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.9\columnwidth]{si-figures/SI-heterotrophs-lolli-05062021.png}
    \caption{Lollipop plot showing the relative CPM of mapped transcripts associated with more phototrophic (left) or heterotrophic (right) traits for 4 SAR TOPAZ MAGs (left to right). Relative abundance is shown by MAG for each sample (y-axis and color bar). TOPAZ MAGs selected are a subset from the SAR clade closely related to stramenopiles and were predicted to be heterotrophic (high H-index; reported next to each TOPAZ MAG name).
    }
    \label{fig:dictyocho-SAR-het-lolli}
\end{figure}
\end{landscape}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/confusionmatrix.png}
    \caption{Confusion matrix generated from Random Forest model construction on the 25\% of the reference transcriptomes used for testing. The $x$ categorical axis is the manually-annotated trophic mode, while the $y$ categorical axis is the predicted trophic mode via the Random Forest model. The greatest number of erroneous predictions involved mixotrophy (8 of 12).}
    \label{fig:confusionmatrix}
\end{figure}

\begin{landscape}
    \begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/burns_categories.png}
    \caption{Comparison of Heterotrophy Index scores with the Burns model prediction for photosynthetic ability. On the $y$ axis, MAGs are split into categories based on their photosynthesis score from the Burns model (which ranges from zero to one): ``complete photosynthesis'' (0.9-1), ``high photosynthesis'' (0.55-0.9), ``mid photosynthesis'' (0.45-0.55), ``low photosynthesis'' (0.1-0.45) and ``no photosynthesis'' (0-0.1).}
    \label{fig:burns-cats}
\end{figure}
\end{landscape}


\begin{landscape}
\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/KO_figure_trophy.png}
    \caption{Presence-absence of KEGG IDs as aggregated across all reference transcriptomes (MMETSP and EukProt) used in the trophic mode modeling process. The top boxplot panel shows the mean presence of the KEGG IDs implicated in the listed pathways for each of the three identified trophic modes, while the bottom heatmap panel shows the average presence of each individual KO along the horizontal axis for reference transcriptomes annotated as each major trophic mode (vertical axis).}
    \label{fig:ko-trophy}
\end{figure}
\end{landscape}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/mmetsp_burns.png}
    \caption{Comparison of the Burns \citep{burns2018gene} model to our Random Forest predictions and Heterotrophy Index calculations for the reference MMETSP transcriptomes. Left: Burns \cite{burns2018gene} photosynthesis predictions vs. composite Heterotrophy Index, scaled by the phagocytosis prediction and colored by the manually-annotated trophic mode. Right: predicted photosynthetic ability by taxonomic grouping, colored by the calculated Heterotrophy Index.}
    \label{fig:mmetsp-burns}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/mag_burns.png}
    \caption{Comparison of the Burns \citep{burns2018gene} model to our Random Forest predictions and Heterotrophy Index calculations for the TOPAZ MAGs. Left: Burns \cite{burns2018gene} photosynthesis predictions vs. composite Heterotrophy Index, scaled by the phagocytosis prediction and shape indicating the Random Forest-derived predicted trophic mode (note there is no color because trophic mode could not be manually annotated). Right: predicted photosynthetic ability by taxonomic grouping, colored by the calculated Heterotrophy Index.}
    \label{fig:mag-burns}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/heterotrophyindex_group_new.png}
    \caption{Heterotrophy Index scores as a function of BUSCO completeness, faceted by taxonomy phylum. The strongest association of Heterotrophy Index with completeness was found within Metazoa, for which it appears that the magnitude of the Heterotrophy Index is tightly coupled to the level of completeness of the MAG. By contrast, Cryptophyta (which are known to be mixotrophic \citep{jones2000mixotrophy}) showed a much weaker coupling of completeness with the magnitude of the Heterotrophy Index, and all values of the Index were closer to zero.}
    \label{fig:het-ind}
\end{figure}


\begin{figure}[h!]    
    \centering
    \includegraphics[width = 0.75\columnwidth]{si-figures/Figure2-TOPAZ_prok_features_vsGORG_mod.png}
    \caption{Comparison of the genomic characteristics between the TOPAZ MAGs and GORG SAGs belonging to the phyla, Bacteroidota, Cyanobacteria and Proteobacteria. The distributions of GC \% content, estimated genome size (Mbp), number of CAZY hits per genome, number of CAZY hits per Mbp per genome, number of peptidase genes per genome and number of peptidase genes per Mbp per genome are presented as box plots. }
    \label{fig:fig6-prokfunc}
\end{figure}


\begin{landscape}
    \begin{figure}
        \centering
        \includegraphics[width=0.95\columnwidth]{si-figures/network_supporting.png}
        \caption{Supporting figures for the network analysis section. A and B: FastANI clustering does not typically group together eukaryotic MAGs of different overall taxonomic classification (A) or taxonomic order (B). C: Justification for ANI cutoff of 95\% (and correlation coefficient 0.504) for considering eukaryotic MAGs to be sufficiently similar to be clustered. The majority of MAGs with pairwise ANI scores of $\geq$95 have correlation coefficients of $\geq$0.8. D: Taxonomic composition of the 7 identified communities from the main text.}
        \label{fig:network-support}
    \end{figure}
\end{landscape}

\begin{landscape}
    \begin{figure}
        \centering
        \includegraphics[width=0.95\columnwidth]{si-figures/bygroup.png}
        \caption{Extended environmental correlations figure displaying the strength and directions of environmental correlations as tracked by both taxonomic group and network-based community.}
        \label{fig:network-bygroup}
    \end{figure}
\end{landscape}

\begin{figure}
    \centering
    \includegraphics[width=0.95\columnwidth]{si-figures/Network_community_maps-01.png}
    \caption{Distribution of identified communities across \textit{Tara} Oceans Samples. The summed metagenomic abundance (CPM) of each of the seven identified communities from the network analysis (Figure 6) is shown for each size fraction. The size of the bubble indicates the relative abundance of the community in a given sample. }
    \label{fig:network-map}
\end{figure}


\begin{figure}[h!]    
    \centering
    \includegraphics[width = 0.75\columnwidth]{si-figures/Heatmap_mod.png}
    \caption{Genomic characteristics of the Community 1 MAGs. MAG names are color-coded based on the taxonomic affiliations estimated by GTDB-tk. The presence of the high oxygen affinity cytochromes db ubiquinol oxidase and cbb3, as well as the reductases (involved in anaerobic processes) DMSO family type II, nitric oxide and nitrite are noted with grey in the left panel of the heat map. The presence of genes involved in CAZy pathways for the hydrolysis/degradation of amorphous cellulose, arabinan, arabinose cleavage, beta-galactan, beta-mannan, chitin, mixed-linkage glucans, pectin, rhamnose cleavage, starch, sulf-polysachharides, xylans, xyloglucan are noted with grey in the right panel of the heat map. The estimated size of the genomes is shown as a bar plot.}
    \label{fig:community1-bact}
\end{figure}

\clearpage


\bibliographystyle{abbrvnat}

\bibliography{main}
\end{document}
